modules/demo-lwjgl/start.sh

```diff
@@ -1,3 +1,7 @@
 #!/bin/sh
-
-java -jar -Xmx2048m target/viz-engine-lwjgl-demo-1.0.0-SNAPSHOT.jar "$@"
+if  [[ "$OSTYPE" == "darwin"* ]]; then
+  # on Macos, ljwgl expect to be launched with -XstartOnFirstThread on jvm
+  java -XstartOnFirstThread -jar -Xmx2048m target/viz-engine-lwjgl-demo-1.0.0-SNAPSHOT.jar "$@"
```

Comment:
I noticed `XstartOnFirstThread` makes AWT not work, so it's only necessary for GLFW

Reply:
On my side it start and crash immediatly (got maybe .01s of a windwos  appears and then disapear) : 
```
-------------------------------------
Translated Report (Full Report Below)
-------------------------------------

Process:               java [25045]
Path:                  /Users/USER/Library/Java/JavaVirtualMachines/openjdk-19/Contents/Home/bin/java
Identifier:            java
Version:               ???
Code Type:             ARM-64 (Native)
Parent Process:        Exited process [25044]
Responsible:           idea [61942]
User ID:               502

Date/Time:             2023-01-05 10:43:58.4209 +0100
OS Version:            macOS 12.5.1 (21G83)
Report Version:        12
Anonymous UUID:        CF12E8EF-4403-4D83-9D3E-BE4B108DB64E

Sleep/Wake UUID:       FD5DA074-D5A6-4648-9990-7A503F8A85E8

Time Awake Since Boot: 130000 seconds
Time Since Wake:       11409 seconds

System Integrity Protection: enabled

Crashed Thread:        36  Java: AWT-EventQueue-0

Exception Type:        EXC_BREAKPOINT (SIGTRAP)
Exception Codes:       0x0000000000000001, 0x000000019d8b5f00
Exception Note:        EXC_CORPSE_NOTIFY

Termination Reason:    Namespace SIGNAL, Code 5 Trace/BPT trap: 5
Terminating Process:   exc handler [25045]

Application Specific Information:
-[NSOpenGLContext setView:] must be called from the main thread.


Thread 0::  Dispatch queue: com.apple.main-thread
0   libsystem_kernel.dylib        	       0x19a9028b0 mach_msg_trap + 8
1   libsystem_kernel.dylib        	       0x19a902d20 mach_msg + 76
2   CoreFoundation                	       0x19aa0d2c0 __CFRunLoopServiceMachPort + 372
3   CoreFoundation                	       0x19aa0b770 __CFRunLoopRun + 1180
4   CoreFoundation                	       0x19aa0ab34 CFRunLoopRunSpecific + 600
5   HIToolbox                     	       0x1a364a338 RunCurrentEventLoopInMode + 292
6   HIToolbox                     	       0x1a364a0b4 ReceiveNextEventCommon + 564
7   HIToolbox                     	       0x1a3649e68 _BlockUntilNextEventMatchingListInModeWithFilter + 72
8   AppKit                        	       0x19d57251c _DPSNextEvent + 860
9   AppKit                        	       0x19d570e14 -[NSApplication(NSEvent) _nextEventMatchingEventMask:untilDate:inMode:dequeue:] + 1328
10  libosxapp.dylib               	       0x105334260 -[NSApplicationAWT nextEventMatchingMask:untilDate:inMode:dequeue:] + 144
11  AppKit                        	       0x19d562fe0 -[NSApplication run] + 596
12  libosxapp.dylib               	       0x105334018 +[NSApplicationAWT runAWTLoopWithApp:] + 208
13  libawt_lwawt.dylib            	       0x107c6ca80 +[AWTStarter starter:headless:] + 488
14  libosxapp.dylib               	       0x105335c90 +[ThreadUtilities invokeBlockCopy:] + 28
15  Foundation                    	       0x19b910f94 __NSThreadPerformPerform + 212
16  CoreFoundation                	       0x19aa0d044 __CFRUNLOOP_IS_CALLING_OUT_TO_A_SOURCE0_PERFORM_FUNCTION__ + 28
17  CoreFoundation                	       0x19aa0cf90 __CFRunLoopDoSource0 + 208
18  CoreFoundation                	       0x19aa0cc90 __CFRunLoopDoSources0 + 268
19  CoreFoundation                	       0x19aa0b610 __CFRunLoopRun + 828
20  CoreFoundation                	       0x19aa0ab34 CFRunLoopRunSpecific + 600
21  libjli.dylib                  	       0x1041235c4 CreateExecutionEnvironment + 400
22  libjli.dylib                  	       0x10411f9c0 JLI_Launch + 1212
23  java                          	       0x1040dfbb4 main + 404
24  dyld                          	       0x1041e108c start + 520

Thread 1:
0   libsystem_kernel.dylib        	       0x19a904834 __ulock_wait + 8
1   libsystem_pthread.dylib       	       0x19a9425a0 _pthread_join + 444
2   libjli.dylib                  	       0x104123c9c CallJavaMainInNewThread + 184
3   libjli.dylib                  	       0x104122b4c ContinueInNewThread + 144
4   libjli.dylib                  	       0x104120a94 JLI_Launch + 5520
5   java                          	       0x1040dfbb4 main + 404
6   libjli.dylib                  	       0x10412455c apple_main + 88
7   libsystem_pthread.dylib       	       0x19a94026c _pthread_start + 148
8   libsystem_pthread.dylib       	       0x19a93b08c thread_start + 8

Thread 2:
0   libsystem_kernel.dylib        	       0x19a906270 __psynch_cvwait + 8
1   libsystem_pthread.dylib       	       0x19a94083c _pthread_cond_wait + 1236
2   libjvm.dylib                  	       0x105bfb528 os::PlatformMonitor::wait(long) + 152
3   libjvm.dylib                  	       0x105bc0868 Monitor::wait(long long) + 128
4   libjvm.dylib                  	       0x105d4f3ec Threads::destroy_vm() + 100
5   libjvm.dylib                  	       0x105915938 jni_DestroyJavaVM + 212
6   libjli.dylib                  	       0x104121bf8 JavaMain + 3156
7   libjli.dylib                  	       0x104123ce4 ThreadJavaMain + 12
8   libsystem_pthread.dylib       	       0x19a94026c _pthread_start + 148
9   libsystem_pthread.dylib       	       0x19a93b08c thread_start + 8

Thread 3:: Java: GC Thread#0
0   libsystem_kernel.dylib        	       0x19a9028ec semaphore_wait_trap + 8
1   libjvm.dylib                  	       0x105c7dd44 OSXSemaphore::wait() + 24
2   libjvm.dylib                  	       0x105dee140 WorkerThread::run() + 84
3   libjvm.dylib                  	       0x105d48288 Thread::call_run() + 224
4   libjvm.dylib                  	       0x105bf3634 thread_native_entry(Thread*) + 344
5   libsystem_pthread.dylib       	       0x19a94026c _pthread_start + 148
6   libsystem_pthread.dylib       	       0x19a93b08c thread_start + 8

Thread 4:: Java: G1 Main Marker
0   libsystem_kernel.dylib        	       0x19a906270 __psynch_cvwait + 8
1   libsystem_pthread.dylib       	       0x19a94083c _pthread_cond_wait + 1236
2   libjvm.dylib                  	       0x105bfb528 os::PlatformMonitor::wait(long) + 152
3   libjvm.dylib                  	       0x105bc07d0 Monitor::wait_without_safepoint_check(long long) + 48
4   libjvm.dylib                  	       0x1057a6e88 G1ConcurrentMarkThread::run_service() + 156
5   libjvm.dylib                  	       0x1056ad75c ConcurrentGCThread::run() + 36
6   libjvm.dylib                  	       0x105d48288 Thread::call_run() + 224
7   libjvm.dylib                  	       0x105bf3634 thread_native_entry(Thread*) + 344
8   libsystem_pthread.dylib       	       0x19a94026c _pthread_start + 148
9   libsystem_pthread.dylib       	       0x19a93b08c thread_start + 8

Thread 5:: Java: G1 Conc#0
0   libsystem_kernel.dylib        	       0x19a9028ec semaphore_wait_trap + 8
1   libjvm.dylib                  	       0x105c7dd44 OSXSemaphore::wait() + 24
2   libjvm.dylib                  	       0x105dee140 WorkerThread::run() + 84
3   libjvm.dylib                  	       0x105d48288 Thread::call_run() + 224
4   libjvm.dylib                  	       0x105bf3634 thread_native_entry(Thread*) + 344
5   libsystem_pthread.dylib       	       0x19a94026c _pthread_start + 148
6   libsystem_pthread.dylib       	       0x19a93b08c thread_start + 8

Thread 6:: Java: G1 Refine#0
0   libsystem_kernel.dylib        	       0x19a9028ec semaphore_wait_trap + 8
1   libjvm.dylib                  	       0x105c7dd44 OSXSemaphore::wait() + 24
2   libjvm.dylib                  	       0x1057a9a48 G1PrimaryConcurrentRefineThread::wait_for_completed_buffers() + 24
3   libjvm.dylib                  	       0x1057a95b4 G1ConcurrentRefineThread::run_service() + 64
4   libjvm.dylib                  	       0x1056ad75c ConcurrentGCThread::run() + 36
5   libjvm.dylib                  	       0x105d48288 Thread::call_run() + 224
6   libjvm.dylib                  	       0x105bf3634 thread_native_entry(Thread*) + 344
7   libsystem_pthread.dylib       	       0x19a94026c _pthread_start + 148
8   libsystem_pthread.dylib       	       0x19a93b08c thread_start + 8

Thread 7:: Java: G1 Service
0   libsystem_kernel.dylib        	       0x19a906270 __psynch_cvwait + 8
1   libsystem_pthread.dylib       	       0x19a94083c _pthread_cond_wait + 1236
2   libjvm.dylib                  	       0x105bfb598 os::PlatformMonitor::wait(long) + 264
3   libjvm.dylib                  	       0x105bc07d0 Monitor::wait_without_safepoint_check(long long) + 48
4   libjvm.dylib                  	       0x1057f30c0 G1ServiceThread::wait_for_task() + 172
5   libjvm.dylib                  	       0x1057f32c0 G1ServiceThread::run_service() + 40
6   libjvm.dylib                  	       0x1056ad75c ConcurrentGCThread::run() + 36
7   libjvm.dylib                  	       0x105d48288 Thread::call_run() + 224
8   libjvm.dylib                  	       0x105bf3634 thread_native_entry(Thread*) + 344
9   libsystem_pthread.dylib       	       0x19a94026c _pthread_start + 148
10  libsystem_pthread.dylib       	       0x19a93b08c thread_start + 8

Thread 8:: Java: VM Thread
0   libsystem_kernel.dylib        	       0x19a906270 __psynch_cvwait + 8
1   libsystem_pthread.dylib       	       0x19a94083c _pthread_cond_wait + 1236
2   libjvm.dylib                  	       0x105bfb598 os::PlatformMonitor::wait(long) + 264
3   libjvm.dylib                  	       0x105bc07d0 Monitor::wait_without_safepoint_check(long long) + 48
4   libjvm.dylib                  	       0x105dc9074 VMThread::wait_for_operation() + 468
5   libjvm.dylib                  	       0x105dc8114 VMThread::run() + 188
6   libjvm.dylib                  	       0x105d48288 Thread::call_run() + 224
7   libjvm.dylib                  	       0x105bf3634 thread_native_entry(Thread*) + 344
8   libsystem_pthread.dylib       	       0x19a94026c _pthread_start + 148
9   libsystem_pthread.dylib       	       0x19a93b08c thread_start + 8

Thread 9:
0   libsystem_pthread.dylib       	       0x19a93b078 start_wqthread + 0

Thread 10:
0   libsystem_pthread.dylib       	       0x19a93b078 start_wqthread + 0

Thread 11:: Java: Reference Handler
0   libsystem_kernel.dylib        	       0x19a906270 __psynch_cvwait + 8
1   libsystem_pthread.dylib       	       0x19a94083c _pthread_cond_wait + 1236
2   libjvm.dylib                  	       0x105bfb528 os::PlatformMonitor::wait(long) + 152
3   libjvm.dylib                  	       0x105bc0868 Monitor::wait(long long) + 128
4   libjvm.dylib                  	       0x105953c10 JVM_WaitForReferencePendingList + 204
5   ???                           	       0x137548aac ???
6   ???                           	       0x137545548 ???
7   ???                           	       0x137545548 ???
8   ???                           	       0x137534140 ???
9   libjvm.dylib                  	       0x10589baf8 JavaCalls::call_helper(JavaValue*, methodHandle const&, JavaCallArguments*, JavaThread*) + 908
10  libjvm.dylib                  	       0x10589ab14 JavaCalls::call_virtual(JavaValue*, Klass*, Symbol*, Symbol*, JavaCallArguments*, JavaThread*) + 284
11  libjvm.dylib                  	       0x10589abe0 JavaCalls::call_virtual(JavaValue*, Handle, Klass*, Symbol*, Symbol*, JavaThread*) + 100
12  libjvm.dylib                  	       0x1059515cc thread_entry(JavaThread*, JavaThread*) + 156
13  libjvm.dylib                  	       0x105d49d34 JavaThread::thread_main_inner() + 272
14  libjvm.dylib                  	       0x105d48288 Thread::call_run() + 224
15  libjvm.dylib                  	       0x105bf3634 thread_native_entry(Thread*) + 344
16  libsystem_pthread.dylib       	       0x19a94026c _pthread_start + 148
17  libsystem_pthread.dylib       	       0x19a93b08c thread_start + 8

Thread 12:: Java: Finalizer
0   libsystem_kernel.dylib        	       0x19a906270 __psynch_cvwait + 8
1   libsystem_pthread.dylib       	       0x19a94083c _pthread_cond_wait + 1236
2   libjvm.dylib                  	       0x105bfa7f4 os::PlatformEvent::park() + 164
3   libjvm.dylib                  	       0x105bdbf58 ObjectMonitor::wait(long, bool, JavaThread*) + 1736
4   libjvm.dylib                  	       0x105d18a7c ObjectSynchronizer::wait(Handle, long, JavaThread*) + 272
5   libjvm.dylib                  	       0x105940824 JVM_MonitorWait + 368
6   ???                           	       0x137548aac ???
7   ???                           	       0x137545548 ???
8   ???                           	       0x137545548 ???
9   ???                           	       0x137545548 ???
10  ???                           	       0x137545548 ???
11  ???                           	       0x137545300 ???
12  ???                           	       0x137545300 ???
13  ???                           	       0x137534140 ???
14  libjvm.dylib                  	       0x10589baf8 JavaCalls::call_helper(JavaValue*, methodHandle const&, JavaCallArguments*, JavaThread*) + 908
15  libjvm.dylib                  	       0x10589ab14 JavaCalls::call_virtual(JavaValue*, Klass*, Symbol*, Symbol*, JavaCallArguments*, JavaThread*) + 284
16  libjvm.dylib                  	       0x10589abe0 JavaCalls::call_virtual(JavaValue*, Handle, Klass*, Symbol*, Symbol*, JavaThread*) + 100
17  libjvm.dylib                  	       0x1059515cc thread_entry(JavaThread*, JavaThread*) + 156
18  libjvm.dylib                  	       0x105d49d34 JavaThread::thread_main_inner() + 272
19  libjvm.dylib                  	       0x105d48288 Thread::call_run() + 224
20  libjvm.dylib                  	       0x105bf3634 thread_native_entry(Thread*) + 344
21  libsystem_pthread.dylib       	       0x19a94026c _pthread_start + 148
22  libsystem_pthread.dylib       	       0x19a93b08c thread_start + 8

Thread 13:: Java: Signal Dispatcher
0   libsystem_kernel.dylib        	       0x19a9028ec semaphore_wait_trap + 8
1   libjvm.dylib                  	       0x105c7dd44 OSXSemaphore::wait() + 24
2   libjvm.dylib                  	       0x105c9268c os::signal_wait() + 180
3   libjvm.dylib                  	       0x105beead8 signal_thread_entry(JavaThread*, JavaThread*) + 76
4   libjvm.dylib                  	       0x105d49d34 JavaThread::thread_main_inner() + 272
5   libjvm.dylib                  	       0x105d48288 Thread::call_run() + 224
6   libjvm.dylib                  	       0x105bf3634 thread_native_entry(Thread*) + 344
7   libsystem_pthread.dylib       	       0x19a94026c _pthread_start + 148
8   libsystem_pthread.dylib       	       0x19a93b08c thread_start + 8

Thread 14:: Java: Service Thread
0   libsystem_kernel.dylib        	       0x19a906270 __psynch_cvwait + 8
1   libsystem_pthread.dylib       	       0x19a94083c _pthread_cond_wait + 1236
2   libjvm.dylib                  	       0x105bfb528 os::PlatformMonitor::wait(long) + 152
3   libjvm.dylib                  	       0x105bc07d0 Monitor::wait_without_safepoint_check(long long) + 48
4   libjvm.dylib                  	       0x105c7e7f4 ServiceThread::service_thread_entry(JavaThread*, JavaThread*) + 484
5   libjvm.dylib                  	       0x105d49d34 JavaThread::thread_main_inner() + 272
6   libjvm.dylib                  	       0x105d48288 Thread::call_run() + 224
7   libjvm.dylib                  	       0x105bf3634 thread_native_entry(Thread*) + 344
8   libsystem_pthread.dylib       	       0x19a94026c _pthread_start + 148
9   libsystem_pthread.dylib       	       0x19a93b08c thread_start + 8

Thread 15:: Java: Monitor Deflation Thread
0   libsystem_kernel.dylib        	       0x19a906270 __psynch_cvwait + 8
1   libsystem_pthread.dylib       	       0x19a94083c _pthread_cond_wait + 1236
2   libjvm.dylib                  	       0x105bfb598 os::PlatformMonitor::wait(long) + 264
3   libjvm.dylib                  	       0x105bc07d0 Monitor::wait_without_safepoint_check(long long) + 48
4   libjvm.dylib                  	       0x105bb5544 MonitorDeflationThread::monitor_deflation_thread_entry(JavaThread*, JavaThread*) + 152
5   libjvm.dylib                  	       0x105d49d34 JavaThread::thread_main_inner() + 272
6   libjvm.dylib                  	       0x105d48288 Thread::call_run() + 224
7   libjvm.dylib                  	       0x105bf3634 thread_native_entry(Thread*) + 344
8   libsystem_pthread.dylib       	       0x19a94026c _pthread_start + 148
9   libsystem_pthread.dylib       	       0x19a93b08c thread_start + 8

Thread 16:: Java: C2 CompilerThread0
0   libjvm.dylib                  	       0x105c33118 PhaseChaitin::elide_copy(Node*, int, Block*, Node_List&, Node_List&, bool) + 1052
1   libjvm.dylib                  	       0x105c32f10 PhaseChaitin::elide_copy(Node*, int, Block*, Node_List&, Node_List&, bool) + 532
2   libjvm.dylib                  	       0x105c3472c PhaseChaitin::post_allocate_copy_removal() + 2984
3   libjvm.dylib                  	       0x105612bdc PhaseChaitin::Register_Allocate() + 1976
4   libjvm.dylib                  	       0x105689d9c Compile::Code_Gen() + 760
5   libjvm.dylib                  	       0x105687abc Compile::Compile(ciEnv*, ciMethod*, int, Options, DirectiveSet*) + 2944
6   libjvm.dylib                  	       0x1055f3a08 C2Compiler::compile_method(ciEnv*, ciMethod*, int, bool, DirectiveSet*) + 308
7   libjvm.dylib                  	       0x105698944 CompileBroker::invoke_compiler_on_method(CompileTask*) + 1280
8   libjvm.dylib                  	       0x1056983ac CompileBroker::compiler_thread_loop() + 1112
9   libjvm.dylib                  	       0x105d49d34 JavaThread::thread_main_inner() + 272
10  libjvm.dylib                  	       0x105d48288 Thread::call_run() + 224
11  libjvm.dylib                  	       0x105bf3634 thread_native_entry(Thread*) + 344
12  libsystem_pthread.dylib       	       0x19a94026c _pthread_start + 148
13  libsystem_pthread.dylib       	       0x19a93b08c thread_start + 8

Thread 17:: Java: C1 CompilerThread0
0   libsystem_kernel.dylib        	       0x19a906270 __psynch_cvwait + 8
1   libsystem_pthread.dylib       	       0x19a94083c _pthread_cond_wait + 1236
2   libjvm.dylib                  	       0x105bfb598 os::PlatformMonitor::wait(long) + 264
3   libjvm.dylib                  	       0x105bc0868 Monitor::wait(long long) + 128
4   libjvm.dylib                  	       0x105694e04 CompileQueue::get(CompilerThread*) + 728
5   libjvm.dylib                  	       0x1056981e8 CompileBroker::compiler_thread_loop() + 660
6   libjvm.dylib                  	       0x105d49d34 JavaThread::thread_main_inner() + 272
7   libjvm.dylib                  	       0x105d48288 Thread::call_run() + 224
8   libjvm.dylib                  	       0x105bf3634 thread_native_entry(Thread*) + 344
9   libsystem_pthread.dylib       	       0x19a94026c _pthread_start + 148
10  libsystem_pthread.dylib       	       0x19a93b08c thread_start + 8

Thread 18:: Java: Sweeper thread
0   libsystem_kernel.dylib        	       0x19a906270 __psynch_cvwait + 8
1   libsystem_pthread.dylib       	       0x19a94083c _pthread_cond_wait + 1236
2   libjvm.dylib                  	       0x105bfb598 os::PlatformMonitor::wait(long) + 264
3   libjvm.dylib                  	       0x105bc07d0 Monitor::wait_without_safepoint_check(long long) + 48
4   libjvm.dylib                  	       0x105d0fdac NMethodSweeper::sweeper_loop() + 148
5   libjvm.dylib                  	       0x105d49d34 JavaThread::thread_main_inner() + 272
6   libjvm.dylib                  	       0x105d48288 Thread::call_run() + 224
7   libjvm.dylib                  	       0x105bf3634 thread_native_entry(Thread*) + 344
8   libsystem_pthread.dylib       	       0x19a94026c _pthread_start + 148
9   libsystem_pthread.dylib       	       0x19a93b08c thread_start + 8

Thread 19:: Java: Notification Thread
0   libsystem_kernel.dylib        	       0x19a906270 __psynch_cvwait + 8
1   libsystem_pthread.dylib       	       0x19a94083c _pthread_cond_wait + 1236
2   libjvm.dylib                  	       0x105bfb528 os::PlatformMonitor::wait(long) + 152
3   libjvm.dylib                  	       0x105bc07d0 Monitor::wait_without_safepoint_check(long long) + 48
4   libjvm.dylib                  	       0x105bd4cb4 NotificationThread::notification_thread_entry(JavaThread*, JavaThread*) + 164
5   libjvm.dylib                  	       0x105d49d34 JavaThread::thread_main_inner() + 272
6   libjvm.dylib                  	       0x105d48288 Thread::call_run() + 224
7   libjvm.dylib                  	       0x105bf3634 thread_native_entry(Thread*) + 344
8   libsystem_pthread.dylib       	       0x19a94026c _pthread_start + 148
9   libsystem_pthread.dylib       	       0x19a93b08c thread_start + 8

Thread 20:: Java: VM Periodic Task Thread
0   libsystem_kernel.dylib        	       0x19a906270 __psynch_cvwait + 8
1   libsystem_pthread.dylib       	       0x19a94083c _pthread_cond_wait + 1236
2   libjvm.dylib                  	       0x105bfb598 os::PlatformMonitor::wait(long) + 264
3   libjvm.dylib                  	       0x105bc07d0 Monitor::wait_without_safepoint_check(long long) + 48
4   libjvm.dylib                  	       0x105bd4640 WatcherThread::sleep() const + 176
5   libjvm.dylib                  	       0x105bd46ec WatcherThread::run() + 76
6   libjvm.dylib                  	       0x105d48288 Thread::call_run() + 224
7   libjvm.dylib                  	       0x105bf3634 thread_native_entry(Thread*) + 344
8   libsystem_pthread.dylib       	       0x19a94026c _pthread_start + 148
9   libsystem_pthread.dylib       	       0x19a93b08c thread_start + 8

Thread 21:: Java: Common-Cleaner
0   libsystem_kernel.dylib        	       0x19a906270 __psynch_cvwait + 8
1   libsystem_pthread.dylib       	       0x19a94083c _pthread_cond_wait + 1236
2   libjvm.dylib                  	       0x105bfadec Parker::park(bool, long) + 484
3   libjvm.dylib                  	       0x105d7ecb8 Unsafe_Park(JNIEnv_*, _jobject*, unsigned char, long) + 336
4   ???                           	       0x137548aac ???
5   ???                           	       0x137545548 ???
6   ???                           	       0x137545548 ???
7   ???                           	       0x137545928 ???
8   ???                           	       0x137545548 ???
9   ???                           	       0x137545300 ???
10  ???                           	       0x137545300 ???
11  ???                           	       0x137545a88 ???
12  ???                           	       0x137545548 ???
13  ???                           	       0x137534140 ???
14  libjvm.dylib                  	       0x10589baf8 JavaCalls::call_helper(JavaValue*, methodHandle const&, JavaCallArguments*, JavaThread*) + 908
15  libjvm.dylib                  	       0x10589ab14 JavaCalls::call_virtual(JavaValue*, Klass*, Symbol*, Symbol*, JavaCallArguments*, JavaThread*) + 284
16  libjvm.dylib                  	       0x10589abe0 JavaCalls::call_virtual(JavaValue*, Handle, Klass*, Symbol*, Symbol*, JavaThread*) + 100
17  libjvm.dylib                  	       0x1059515cc thread_entry(JavaThread*, JavaThread*) + 156
18  libjvm.dylib                  	       0x105d49d34 JavaThread::thread_main_inner() + 272
19  libjvm.dylib                  	       0x105d48288 Thread::call_run() + 224
20  libjvm.dylib                  	       0x105bf3634 thread_native_entry(Thread*) + 344
21  libsystem_pthread.dylib       	       0x19a94026c _pthread_start + 148
22  libsystem_pthread.dylib       	       0x19a93b08c thread_start + 8

Thread 22::  Dispatch queue: com.apple.root.utility-qos
0   dyld                          	       0x1041f7168 dyld4::PrebuiltLoader::contains(dyld4::RuntimeState&, void const*, void const**, unsigned long long*, unsigned char*) const + 116
1   dyld                          	       0x1041f7124 dyld4::PrebuiltLoader::contains(dyld4::RuntimeState&, void const*, void const**, unsigned long long*, unsigned char*) const + 48
2   dyld                          	       0x104201918 invocation function for block in dyld4::APIs::findImageContaining(void*) + 124
3   dyld                          	       0x1041e6a98 dyld4::RuntimeState::withLoadersReadLock(void () block_pointer) + 92
4   dyld                          	       0x10420096c dyld4::APIs::findImageContaining(void*) + 132
5   dyld                          	       0x104201b5c dyld4::APIs::dlopen_from(char const*, int, void*) + 256
6   OpenGL                        	       0x1fc0c9b58 0x1fc0bb000 + 60248
7   libdispatch.dylib             	       0x19a77c1b4 _dispatch_client_callout + 20
8   libdispatch.dylib             	       0x19a77da34 _dispatch_once_callout + 32
9   OpenGL                        	       0x1fc0c9ab0 0x1fc0bb000 + 60080
10  OpenGL                        	       0x1fc0c991c 0x1fc0bb000 + 59676
11  libdispatch.dylib             	       0x19a77a5f0 _dispatch_call_block_and_release + 32
12  libdispatch.dylib             	       0x19a77c1b4 _dispatch_client_callout + 20
13  libdispatch.dylib             	       0x19a78da04 _dispatch_root_queue_drain + 680
14  libdispatch.dylib             	       0x19a78e104 _dispatch_worker_thread2 + 164
15  libsystem_pthread.dylib       	       0x19a93c324 _pthread_wqthread + 228
16  libsystem_pthread.dylib       	       0x19a93b080 start_wqthread + 8

Thread 23:: Java: AWT-Shutdown
0   libsystem_kernel.dylib        	       0x19a906270 __psynch_cvwait + 8
1   libsystem_pthread.dylib       	       0x19a94083c _pthread_cond_wait + 1236
2   libjvm.dylib                  	       0x105bfa7f4 os::PlatformEvent::park() + 164
3   libjvm.dylib                  	       0x105bdbf58 ObjectMonitor::wait(long, bool, JavaThread*) + 1736
4   libjvm.dylib                  	       0x105d18a7c ObjectSynchronizer::wait(Handle, long, JavaThread*) + 272
5   libjvm.dylib                  	       0x105940824 JVM_MonitorWait + 368
6   ???                           	       0x137548aac ???
7   ???                           	       0x137545548 ???
8   ???                           	       0x137545548 ???
9   ???                           	       0x137545548 ???
10  ???                           	       0x137545a88 ???
11  ???                           	       0x137534140 ???
12  libjvm.dylib                  	       0x10589baf8 JavaCalls::call_helper(JavaValue*, methodHandle const&, JavaCallArguments*, JavaThread*) + 908
13  libjvm.dylib                  	       0x10589ab14 JavaCalls::call_virtual(JavaValue*, Klass*, Symbol*, Symbol*, JavaCallArguments*, JavaThread*) + 284
14  libjvm.dylib                  	       0x10589abe0 JavaCalls::call_virtual(JavaValue*, Handle, Klass*, Symbol*, Symbol*, JavaThread*) + 100
15  libjvm.dylib                  	       0x1059515cc thread_entry(JavaThread*, JavaThread*) + 156
16  libjvm.dylib                  	       0x105d49d34 JavaThread::thread_main_inner() + 272
17  libjvm.dylib                  	       0x105d48288 Thread::call_run() + 224
18  libjvm.dylib                  	       0x105bf3634 thread_native_entry(Thread*) + 344
19  libsystem_pthread.dylib       	       0x19a94026c _pthread_start + 148
20  libsystem_pthread.dylib       	       0x19a93b08c thread_start + 8

Thread 24:: com.apple.NSEventThread
0   libsystem_kernel.dylib        	       0x19a9028b0 mach_msg_trap + 8
1   libsystem_kernel.dylib        	       0x19a902d20 mach_msg + 76
2   CoreFoundation                	       0x19aa0d2c0 __CFRunLoopServiceMachPort + 372
3   CoreFoundation                	       0x19aa0b770 __CFRunLoopRun + 1180
4   CoreFoundation                	       0x19aa0ab34 CFRunLoopRunSpecific + 600
5   AppKit                        	       0x19d6df100 _NSEventThread + 196
6   libsystem_pthread.dylib       	       0x19a94026c _pthread_start + 148
7   libsystem_pthread.dylib       	       0x19a93b08c thread_start + 8

Thread 25:: Java: Java2D Queue Flusher
0   libsystem_kernel.dylib        	       0x19a906270 __psynch_cvwait + 8
1   libsystem_pthread.dylib       	       0x19a94083c _pthread_cond_wait + 1236
2   libjvm.dylib                  	       0x105bfaa38 os::PlatformEvent::park(long) + 428
3   libjvm.dylib                  	       0x105bdbd80 ObjectMonitor::wait(long, bool, JavaThread*) + 1264
4   libjvm.dylib                  	       0x105d18a7c ObjectSynchronizer::wait(Handle, long, JavaThread*) + 272
5   libjvm.dylib                  	       0x105940824 JVM_MonitorWait + 368
6   ???                           	       0x137548aac ???
7   ???                           	       0x137545548 ???
8   ???                           	       0x137545548 ???
9   ???                           	       0x137545a88 ???
10  ???                           	       0x137534140 ???
11  libjvm.dylib                  	       0x10589baf8 JavaCalls::call_helper(JavaValue*, methodHandle const&, JavaCallArguments*, JavaThread*) + 908
12  libjvm.dylib                  	       0x10589ab14 JavaCalls::call_virtual(JavaValue*, Klass*, Symbol*, Symbol*, JavaCallArguments*, JavaThread*) + 284
13  libjvm.dylib                  	       0x10589abe0 JavaCalls::call_virtual(JavaValue*, Handle, Klass*, Symbol*, Symbol*, JavaThread*) + 100
14  libjvm.dylib                  	       0x1059515cc thread_entry(JavaThread*, JavaThread*) + 156
15  libjvm.dylib                  	       0x105d49d34 JavaThread::thread_main_inner() + 272
16  libjvm.dylib                  	       0x105d48288 Thread::call_run() + 224
17  libjvm.dylib                  	       0x105bf3634 thread_native_entry(Thread*) + 344
18  libsystem_pthread.dylib       	       0x19a94026c _pthread_start + 148
19  libsystem_pthread.dylib       	       0x19a93b08c thread_start + 8

Thread 26:: Java: Java2D Disposer
0   libsystem_kernel.dylib        	       0x19a906270 __psynch_cvwait + 8
1   libsystem_pthread.dylib       	       0x19a94083c _pthread_cond_wait + 1236
2   libjvm.dylib                  	       0x105bfae00 Parker::park(bool, long) + 504
3   libjvm.dylib                  	       0x105d7ecb8 Unsafe_Park(JNIEnv_*, _jobject*, unsigned char, long) + 336
4   ???                           	       0x137548aac ???
5   ???                           	       0x137545548 ???
6   ???                           	       0x137545548 ???
7   ???                           	       0x137545928 ???
8   ???                           	       0x137545548 ???
9   ???                           	       0x137545548 ???
10  ???                           	       0x137545a88 ???
11  ???                           	       0x137545548 ???
12  ???                           	       0x137545300 ???
13  ???                           	       0x137545300 ???
14  ???                           	       0x137545a88 ???
15  ???                           	       0x137534140 ???
16  libjvm.dylib                  	       0x10589baf8 JavaCalls::call_helper(JavaValue*, methodHandle const&, JavaCallArguments*, JavaThread*) + 908
17  libjvm.dylib                  	       0x10589ab14 JavaCalls::call_virtual(JavaValue*, Klass*, Symbol*, Symbol*, JavaCallArguments*, JavaThread*) + 284
18  libjvm.dylib                  	       0x10589abe0 JavaCalls::call_virtual(JavaValue*, Handle, Klass*, Symbol*, Symbol*, JavaThread*) + 100
19  libjvm.dylib                  	       0x1059515cc thread_entry(JavaThread*, JavaThread*) + 156
20  libjvm.dylib                  	       0x105d49d34 JavaThread::thread_main_inner() + 272
21  libjvm.dylib                  	       0x105d48288 Thread::call_run() + 224
22  libjvm.dylib                  	       0x105bf3634 thread_native_entry(Thread*) + 344
23  libsystem_pthread.dylib       	       0x19a94026c _pthread_start + 148
24  libsystem_pthread.dylib       	       0x19a93b08c thread_start + 8

Thread 27:: Java: Active Reference Queue Daemon
0   libsystem_kernel.dylib        	       0x19a906270 __psynch_cvwait + 8
1   libsystem_pthread.dylib       	       0x19a94083c _pthread_cond_wait + 1236
2   libjvm.dylib                  	       0x105bfae00 Parker::park(bool, long) + 504
3   libjvm.dylib                  	       0x105d7ecb8 Unsafe_Park(JNIEnv_*, _jobject*, unsigned char, long) + 336
4   ???                           	       0x137548aac ???
5   ???                           	       0x137545548 ???
6   ???                           	       0x137545548 ???
7   ???                           	       0x137545928 ???
8   ???                           	       0x137545548 ???
9   ???                           	       0x137545548 ???
10  ???                           	       0x137545a88 ???
11  ???                           	       0x137545548 ???
12  ???                           	       0x137545300 ???
13  ???                           	       0x137545300 ???
14  ???                           	       0x137545300 ???
15  ???                           	       0x137534140 ???
16  libjvm.dylib                  	       0x10589baf8 JavaCalls::call_helper(JavaValue*, methodHandle const&, JavaCallArguments*, JavaThread*) + 908
17  libjvm.dylib                  	       0x10589ab14 JavaCalls::call_virtual(JavaValue*, Klass*, Symbol*, Symbol*, JavaCallArguments*, JavaThread*) + 284
18  libjvm.dylib                  	       0x10589abe0 JavaCalls::call_virtual(JavaValue*, Handle, Klass*, Symbol*, Symbol*, JavaThread*) + 100
19  libjvm.dylib                  	       0x1059515cc thread_entry(JavaThread*, JavaThread*) + 156
20  libjvm.dylib                  	       0x105d49d34 JavaThread::thread_main_inner() + 272
21  libjvm.dylib                  	       0x105d48288 Thread::call_run() + 224
22  libjvm.dylib                  	       0x105bf3634 thread_native_entry(Thread*) + 344
23  libsystem_pthread.dylib       	       0x19a94026c _pthread_start + 148
24  libsystem_pthread.dylib       	       0x19a93b08c thread_start + 8

Thread 28:: Java: RequestProcessor queue manager
0   libsystem_kernel.dylib        	       0x19a906270 __psynch_cvwait + 8
1   libsystem_pthread.dylib       	       0x19a94083c _pthread_cond_wait + 1236
2   libjvm.dylib                  	       0x105bfaa38 os::PlatformEvent::park(long) + 428
3   libjvm.dylib                  	       0x105bdbd80 ObjectMonitor::wait(long, bool, JavaThread*) + 1264
4   libjvm.dylib                  	       0x105d18a7c ObjectSynchronizer::wait(Handle, long, JavaThread*) + 272
5   libjvm.dylib                  	       0x105940824 JVM_MonitorWait + 368
6   ???                           	       0x137548aac ???
7   ???                           	       0x137545548 ???
8   ???                           	       0x137545548 ???
9   ???                           	       0x137545300 ???
10  ???                           	       0x137534140 ???
11  libjvm.dylib                  	       0x10589baf8 JavaCalls::call_helper(JavaValue*, methodHandle const&, JavaCallArguments*, JavaThread*) + 908
12  libjvm.dylib                  	       0x10589ab14 JavaCalls::call_virtual(JavaValue*, Klass*, Symbol*, Symbol*, JavaCallArguments*, JavaThread*) + 284
13  libjvm.dylib                  	       0x10589abe0 JavaCalls::call_virtual(JavaValue*, Handle, Klass*, Symbol*, Symbol*, JavaThread*) + 100
14  libjvm.dylib                  	       0x1059515cc thread_entry(JavaThread*, JavaThread*) + 156
15  libjvm.dylib                  	       0x105d49d34 JavaThread::thread_main_inner() + 272
16  libjvm.dylib                  	       0x105d48288 Thread::call_run() + 224
17  libjvm.dylib                  	       0x105bf3634 thread_native_entry(Thread*) + 344
18  libsystem_pthread.dylib       	       0x19a94026c _pthread_start + 148
19  libsystem_pthread.dylib       	       0x19a93b08c thread_start + 8

Thread 29:: Java: GC Thread#1
0   libsystem_kernel.dylib        	       0x19a9028ec semaphore_wait_trap + 8
1   libjvm.dylib                  	       0x105c7dd44 OSXSemaphore::wait() + 24
2   libjvm.dylib                  	       0x105dee140 WorkerThread::run() + 84
3   libjvm.dylib                  	       0x105d48288 Thread::call_run() + 224
4   libjvm.dylib                  	       0x105bf3634 thread_native_entry(Thread*) + 344
5   libsystem_pthread.dylib       	       0x19a94026c _pthread_start + 148
6   libsystem_pthread.dylib       	       0x19a93b08c thread_start + 8

Thread 30:: Java: GC Thread#2
0   libsystem_kernel.dylib        	       0x19a9028ec semaphore_wait_trap + 8
1   libjvm.dylib                  	       0x105c7dd44 OSXSemaphore::wait() + 24
2   libjvm.dylib                  	       0x105dee140 WorkerThread::run() + 84
3   libjvm.dylib                  	       0x105d48288 Thread::call_run() + 224
4   libjvm.dylib                  	       0x105bf3634 thread_native_entry(Thread*) + 344
5   libsystem_pthread.dylib       	       0x19a94026c _pthread_start + 148
6   libsystem_pthread.dylib       	       0x19a93b08c thread_start + 8

Thread 31:: Java: GC Thread#3
0   libsystem_kernel.dylib        	       0x19a9028ec semaphore_wait_trap + 8
1   libjvm.dylib                  	       0x105c7dd44 OSXSemaphore::wait() + 24
2   libjvm.dylib                  	       0x105dee140 WorkerThread::run() + 84
3   libjvm.dylib                  	       0x105d48288 Thread::call_run() + 224
4   libjvm.dylib                  	       0x105bf3634 thread_native_entry(Thread*) + 344
5   libsystem_pthread.dylib       	       0x19a94026c _pthread_start + 148
6   libsystem_pthread.dylib       	       0x19a93b08c thread_start + 8

Thread 32:: Java: GC Thread#4
0   libsystem_kernel.dylib        	       0x19a9028ec semaphore_wait_trap + 8
1   libjvm.dylib                  	       0x105c7dd44 OSXSemaphore::wait() + 24
2   libjvm.dylib                  	       0x105dee140 WorkerThread::run() + 84
3   libjvm.dylib                  	       0x105d48288 Thread::call_run() + 224
4   libjvm.dylib                  	       0x105bf3634 thread_native_entry(Thread*) + 344
5   libsystem_pthread.dylib       	       0x19a94026c _pthread_start + 148
6   libsystem_pthread.dylib       	       0x19a93b08c thread_start + 8

Thread 33:: Java: GC Thread#5
0   libsystem_kernel.dylib        	       0x19a9028ec semaphore_wait_trap + 8
1   libjvm.dylib                  	       0x105c7dd44 OSXSemaphore::wait() + 24
2   libjvm.dylib                  	       0x105dee140 WorkerThread::run() + 84
3   libjvm.dylib                  	       0x105d48288 Thread::call_run() + 224
4   libjvm.dylib                  	       0x105bf3634 thread_native_entry(Thread*) + 344
5   libsystem_pthread.dylib       	       0x19a94026c _pthread_start + 148
6   libsystem_pthread.dylib       	       0x19a93b08c thread_start + 8

Thread 34:: Java: GC Thread#6
0   libsystem_kernel.dylib        	       0x19a9028ec semaphore_wait_trap + 8
1   libjvm.dylib                  	       0x105c7dd44 OSXSemaphore::wait() + 24
2   libjvm.dylib                  	       0x105dee140 WorkerThread::run() + 84
3   libjvm.dylib                  	       0x105d48288 Thread::call_run() + 224
4   libjvm.dylib                  	       0x105bf3634 thread_native_entry(Thread*) + 344
5   libsystem_pthread.dylib       	       0x19a94026c _pthread_start + 148
6   libsystem_pthread.dylib       	       0x19a93b08c thread_start + 8

Thread 35:: Java: GC Thread#7
0   libsystem_kernel.dylib        	       0x19a9028ec semaphore_wait_trap + 8
1   libjvm.dylib                  	       0x105c7dd44 OSXSemaphore::wait() + 24
2   libjvm.dylib                  	       0x105dee140 WorkerThread::run() + 84
3   libjvm.dylib                  	       0x105d48288 Thread::call_run() + 224
4   libjvm.dylib                  	       0x105bf3634 thread_native_entry(Thread*) + 344
5   libsystem_pthread.dylib       	       0x19a94026c _pthread_start + 148
6   libsystem_pthread.dylib       	       0x19a93b08c thread_start + 8

Thread 36 Crashed:: Java: AWT-EventQueue-0
0   AppKit                        	       0x19d8b5f00 -[NSOpenGLContext setView:] + 228
1   AppKit                        	       0x19d8b5f00 -[NSOpenGLContext setView:] + 228
2   AppKit                        	       0x19dc3c500 -[_NSOpenGLViewBackingLayer display] + 628
3   QuartzCore                    	       0x1a18a5f30 CA::Layer::display_if_needed(CA::Transaction*) + 780
4   QuartzCore                    	       0x1a19f262c CA::Context::commit_transaction(CA::Transaction*, double, double*) + 464
5   QuartzCore                    	       0x1a18874cc CA::Transaction::commit() + 704
6   ???                           	       0x137548aac ???
7   ???                           	       0x137545440 ???
8   ???                           	       0x137545548 ???
9   ???                           	       0x137545980 ???
10  ???                           	       0x137545548 ???
11  ???                           	       0x137545548 ???
12  ???                           	       0x137545a88 ???
13  ???                           	       0x137545a88 ???
14  ???                           	       0x137545548 ???
15  ???                           	       0x137545300 ???
16  ???                           	       0x130198964 ???
17  ???                           	       0x137545300 ???
18  ???                           	       0x137545840 ???
19  ???                           	       0x137545548 ???
20  ???                           	       0x137545548 ???
21  ???                           	       0x137545548 ???
22  ???                           	       0x137545548 ???
23  ???                           	       0x137545548 ???
24  ???                           	       0x137545548 ???
25  ???                           	       0x137534140 ???
26  libjvm.dylib                  	       0x10589baf8 JavaCalls::call_helper(JavaValue*, methodHandle const&, JavaCallArguments*, JavaThread*) + 908
27  libjvm.dylib                  	       0x10589ab14 JavaCalls::call_virtual(JavaValue*, Klass*, Symbol*, Symbol*, JavaCallArguments*, JavaThread*) + 284
28  libjvm.dylib                  	       0x10589abe0 JavaCalls::call_virtual(JavaValue*, Handle, Klass*, Symbol*, Symbol*, JavaThread*) + 100
29  libjvm.dylib                  	       0x1059515cc thread_entry(JavaThread*, JavaThread*) + 156
30  libjvm.dylib                  	       0x105d49d34 JavaThread::thread_main_inner() + 272
31  libjvm.dylib                  	       0x105d48288 Thread::call_run() + 224
32  libjvm.dylib                  	       0x105bf3634 thread_native_entry(Thread*) + 344
33  libsystem_pthread.dylib       	       0x19a94026c _pthread_start + 148
34  libsystem_pthread.dylib       	       0x19a93b08c thread_start + 8

Thread 37:
0   libsystem_pthread.dylib       	       0x19a93b078 start_wqthread + 0

Thread 38:
0   libsystem_pthread.dylib       	       0x19a93b078 start_wqthread + 0


Thread 36 crashed with ARM Thread State (64-bit):
    x0: 0x0000000000000000   x1: 0x0000000000000000   x2: 0x0000000000000000   x3: 0x0000000000000000
    x4: 0x0000000000000000   x5: 0x0000000000000000   x6: 0x0000000000000000   x7: 0x0000000000000000
    x8: 0x2ee502b9e3d600b6   x9: 0x2ee502b9e3d600b6  x10: 0x000000001e000000  x11: 0x0000000000000001
   x12: 0x0000000281b04532  x13: 0x0000000000000000  x14: 0x00000001040e8121  x15: 0x00000001f339d450
   x16: 0xfffffffffffffff4  x17: 0x00000001f4a63f00  x18: 0x0000000000000000  x19: 0x00006000032efa00
   x20: 0x0000000105111940  x21: 0x00000001e4d216ea  x22: 0x00006000032efa00  x23: 0x00000002818047e0
   x24: 0x00000001f0c21000  x25: 0x00000001f0c1f000  x26: 0x0000000281b04810  x27: 0x00000001e4cfdadb
   x28: 0x0000000000000001   fp: 0x0000000281b04780   lr: 0xa84080019d8b5f00
    sp: 0x0000000281b04730   pc: 0x000000019d8b5f00 cpsr: 0x60001000
   far: 0x0000000285560000  esr: 0xf2000001 (Breakpoint) brk 1

Binary Images:
       0x19a901000 -        0x19a938fff libsystem_kernel.dylib (*) <d5cdde7e-7037-3a70-9ce4-9e3847da6ce0> /usr/lib/system/libsystem_kernel.dylib
       0x19a988000 -        0x19aecefff com.apple.CoreFoundation (6.9) <1643f3d9-7049-37e4-b844-c0c0a42bde3b> /System/Library/Frameworks/CoreFoundation.framework/Versions/A/CoreFoundation
       0x1a3618000 -        0x1a394bfff com.apple.HIToolbox (2.1.1) <7112f6f2-a77e-39db-83c8-f627b8a49df8> /System/Library/Frameworks/Carbon.framework/Versions/A/Frameworks/HIToolbox.framework/Versions/A/HIToolbox
       0x19d531000 -        0x19e3e9fff com.apple.AppKit (6.9) <07bc3607-552a-3eb9-a941-4911c589dbc5> /System/Library/Frameworks/AppKit.framework/Versions/C/AppKit
       0x105330000 -        0x105337fff libosxapp.dylib (*) <856e2938-924e-3a37-ba51-f4e36166890e> /Users/USER/Library/Java/JavaVirtualMachines/openjdk-19/Contents/Home/lib/libosxapp.dylib
       0x107c00000 -        0x107cbbfff libawt_lwawt.dylib (*) <ddd1182a-2838-3f1b-827b-0f71cd65f46b> /Users/USER/Library/Java/JavaVirtualMachines/openjdk-19/Contents/Home/lib/libawt_lwawt.dylib
       0x19b88c000 -        0x19bc7dfff com.apple.Foundation (6.9) <59ff845a-05b2-3b64-ae06-6ef1d6aee83e> /System/Library/Frameworks/Foundation.framework/Versions/C/Foundation
       0x104118000 -        0x10412bfff libjli.dylib (*) <8f8cedd4-4d12-3aec-9e5e-8d26fd66b393> /Users/USER/Library/Java/JavaVirtualMachines/openjdk-19/Contents/Home/lib/libjli.dylib
       0x1040dc000 -        0x1040dffff java (*) <46d333fe-7faa-3cc8-8390-eef50d4a85f3> /Users/USER/Library/Java/JavaVirtualMachines/openjdk-19/Contents/Home/bin/java
       0x1041dc000 -        0x10423bfff dyld (*) <75627683-a780-32ad-ae34-cf86dd23a26b> /usr/lib/dyld
       0x19a939000 -        0x19a945fff libsystem_pthread.dylib (*) <f32ff902-ba43-30b0-ad43-a2a8a9ff69fe> /usr/lib/system/libsystem_pthread.dylib
       0x1053b8000 -        0x105eebfff libjvm.dylib (*) <02462c0d-ed7e-3b56-a84b-830fc2e1b189> /Users/USER/Library/Java/JavaVirtualMachines/openjdk-19/Contents/Home/lib/server/libjvm.dylib
               0x0 - 0xffffffffffffffff ??? (*) <00000000-0000-0000-0000-000000000000> ???
       0x1fc0bb000 -        0x1fc0cbfff com.apple.opengl (19.5.1) <f538e7dd-90fa-3ea1-ae05-3db879ca7c6a> /System/Library/Frameworks/OpenGL.framework/Versions/A/OpenGL
       0x19a778000 -        0x19a7befff libdispatch.dylib (*) <035d23d9-4cb5-3759-9059-1f7878f89fee> /usr/lib/system/libdispatch.dylib
       0x1a1885000 -        0x1a1bb0fff com.apple.QuartzCore (1.11) <05d8e758-d3ea-3942-bbe3-668a95692658> /System/Library/Frameworks/QuartzCore.framework/Versions/A/QuartzCore

External Modification Summary:
  Calls made by other processes targeting this process:
    task_for_pid: 0
    thread_create: 0
    thread_set_state: 0
  Calls made by this process:
    task_for_pid: 0
    thread_create: 0
    thread_set_state: 0
  Calls made by all processes on this machine:
    task_for_pid: 0
    thread_create: 0
    thread_set_state: 0

VM Region Summary:
ReadOnly portion of Libraries: Total=816.8M resident=0K(0%) swapped_out_or_unallocated=816.8M(100%)
Writable regions: Total=2.0G written=0K(0%) resident=0K(0%) swapped_out=0K(0%) unallocated=2.0G(100%)

                                VIRTUAL   REGION 
REGION TYPE                        SIZE    COUNT (non-coalesced) 
===========                     =======  ======= 
Accelerate framework               512K        4 
Activity Tracing                   256K        1 
CG backing stores                 4800K        8 
CG image                          8304K        4 
ColorSync                          608K       26 
CoreAnimation                      256K        9 
CoreGraphics                        48K        3 
CoreUI image data                 1104K        7 
Foundation                         112K        2 
Image IO                          1024K        1 
Kernel Alloc Once                   32K        1 
MALLOC                           322.6M       57 
MALLOC guard page                  192K       10 
MALLOC_MEDIUM (reserved)         944.0M        8         reserved VM address space (unallocated)
MALLOC_NANO (reserved)           384.0M        1         reserved VM address space (unallocated)
STACK GUARD                       1200K       39 
Stack                             73.8M       39 
Stack Guard                       56.3M       17 
VM_ALLOCATE                        3.4G      339 
VM_ALLOCATE (reserved)            42.0M        9         reserved VM address space (unallocated)
__AUTH                            1819K      160 
__AUTH_CONST                      10.0M      312 
__CTF                               756        1 
__DATA                            10.3M      320 
__DATA_CONST                      11.3M      332 
__DATA_DIRTY                       604K      103 
__FONT_DATA                          4K        1 
__GLSLBUILTINS                    5176K        1 
__LINKEDIT                       582.8M       23 
__OBJC_CONST                      1290K      136 
__OBJC_RO                         83.0M        1 
__OBJC_RW                         3168K        1 
__TEXT                           234.0M      348 
__UNICODE                          592K        1 
dyld private memory               1024K        1 
mapped file                      298.0M       29 
shared memory                      912K       16 
===========                     =======  ======= 
TOTAL                              6.4G     2371 
TOTAL, minus reserved VM space     5.0G     2371 



-----------
Full Report
-----------

{"app_name":"java","timestamp":"2023-01-05 10:43:58.00 +0100","app_version":"","slice_uuid":"46d333fe-7faa-3cc8-8390-eef50d4a85f3","build_version":"","platform":1,"share_with_app_devs":0,"is_first_party":1,"bug_type":"309","os_version":"macOS 12.5.1 (21G83)","incident_id":"07E9904F-4B72-43DD-99BC-7BB074319D8D","name":"java"}
{
  "uptime" : 130000,
  "procLaunch" : "2023-01-05 10:43:56.8540 +0100",
  "procRole" : "Foreground",
  "version" : 2,
  "userID" : 502,
  "deployVersion" : 210,
  "modelCode" : "MacBookPro18,1",
  "procStartAbsTime" : 3282464836451,
  "coalitionID" : 9355,
  "osVersion" : {
    "train" : "macOS 12.5.1",
    "build" : "21G83",
    "releaseType" : "User"
  },
  "captureTime" : "2023-01-05 10:43:58.4209 +0100",
  "incident" : "07E9904F-4B72-43DD-99BC-7BB074319D8D",
  "bug_type" : "309",
  "pid" : 25045,
  "procExitAbsTime" : 3282501485664,
  "translated" : false,
  "cpuType" : "ARM-64",
  "procName" : "java",
  "procPath" : "\/Users\/USER\/Library\/Java\/JavaVirtualMachines\/openjdk-19\/Contents\/Home\/bin\/java",
  "parentProc" : "Exited process",
  "parentPid" : 25044,
  "coalitionName" : "com.jetbrains.intellij.ce",
  "crashReporterKey" : "CF12E8EF-4403-4D83-9D3E-BE4B108DB64E",
  "responsiblePid" : 61942,
  "responsibleProc" : "idea",
  "wakeTime" : 11409,
  "sleepWakeUUID" : "FD5DA074-D5A6-4648-9990-7A503F8A85E8",
  "sip" : "enabled",
  "isCorpse" : 1,
  "exception" : {"codes":"0x0000000000000001, 0x000000019d8b5f00","rawCodes":[1,6938124032],"type":"EXC_BREAKPOINT","signal":"SIGTRAP"},
  "termination" : {"flags":0,"code":5,"namespace":"SIGNAL","indicator":"Trace\/BPT trap: 5","byProc":"exc handler","byPid":25045},
  "asi" : {"libsystem_c.dylib":["-[NSOpenGLContext setView:] must be called from the main thread."]},
  "extMods" : {"caller":{"thread_create":0,"thread_set_state":0,"task_for_pid":0},"system":{"thread_create":0,"thread_set_state":0,"task_for_pid":0},"targeted":{"thread_create":0,"thread_set_state":0,"task_for_pid":0},"warnings":0},
  "faultingThread" : 36,
  "threads" : [{"id":1766446,"queue":"com.apple.main-thread","frames":[{"imageOffset":6320,"symbol":"mach_msg_trap","symbolLocation":8,"imageIndex":0},{"imageOffset":7456,"symbol":"mach_msg","symbolLocation":76,"imageIndex":0},{"imageOffset":545472,"symbol":"__CFRunLoopServiceMachPort","symbolLocation":372,"imageIndex":1},{"imageOffset":538480,"symbol":"__CFRunLoopRun","symbolLocation":1180,"imageIndex":1},{"imageOffset":535348,"symbol":"CFRunLoopRunSpecific","symbolLocation":600,"imageIndex":1},{"imageOffset":205624,"symbol":"RunCurrentEventLoopInMode","symbolLocation":292,"imageIndex":2},{"imageOffset":204980,"symbol":"ReceiveNextEventCommon","symbolLocation":564,"imageIndex":2},{"imageOffset":204392,"symbol":"_BlockUntilNextEventMatchingListInModeWithFilter","symbolLocation":72,"imageIndex":2},{"imageOffset":267548,"symbol":"_DPSNextEvent","symbolLocation":860,"imageIndex":3},{"imageOffset":261652,"symbol":"-[NSApplication(NSEvent) _nextEventMatchingEventMask:untilDate:inMode:dequeue:]","symbolLocation":1328,"imageIndex":3},{"imageOffset":16992,"symbol":"-[NSApplicationAWT nextEventMatchingMask:untilDate:inMode:dequeue:]","symbolLocation":144,"imageIndex":4},{"imageOffset":204768,"symbol":"-[NSApplication run]","symbolLocation":596,"imageIndex":3},{"imageOffset":16408,"symbol":"+[NSApplicationAWT runAWTLoopWithApp:]","symbolLocation":208,"imageIndex":4},{"imageOffset":445056,"symbol":"+[AWTStarter starter:headless:]","symbolLocation":488,"imageIndex":5},{"imageOffset":23696,"symbol":"+[ThreadUtilities invokeBlockCopy:]","symbolLocation":28,"imageIndex":4},{"imageOffset":544660,"symbol":"__NSThreadPerformPerform","symbolLocation":212,"imageIndex":6},{"imageOffset":544836,"symbol":"__CFRUNLOOP_IS_CALLING_OUT_TO_A_SOURCE0_PERFORM_FUNCTION__","symbolLocation":28,"imageIndex":1},{"imageOffset":544656,"symbol":"__CFRunLoopDoSource0","symbolLocation":208,"imageIndex":1},{"imageOffset":543888,"symbol":"__CFRunLoopDoSources0","symbolLocation":268,"imageIndex":1},{"imageOffset":538128,"symbol":"__CFRunLoopRun","symbolLocation":828,"imageIndex":1},{"imageOffset":535348,"symbol":"CFRunLoopRunSpecific","symbolLocation":600,"imageIndex":1},{"imageOffset":46532,"symbol":"CreateExecutionEnvironment","symbolLocation":400,"imageIndex":7},{"imageOffset":31168,"symbol":"JLI_Launch","symbolLocation":1212,"imageIndex":7},{"imageOffset":15284,"symbol":"main","symbolLocation":404,"imageIndex":8},{"imageOffset":20620,"symbol":"start","symbolLocation":520,"imageIndex":9}]},{"id":1766447,"frames":[{"imageOffset":14388,"symbol":"__ulock_wait","symbolLocation":8,"imageIndex":0},{"imageOffset":38304,"symbol":"_pthread_join","symbolLocation":444,"imageIndex":10},{"imageOffset":48284,"symbol":"CallJavaMainInNewThread","symbolLocation":184,"imageIndex":7},{"imageOffset":43852,"symbol":"ContinueInNewThread","symbolLocation":144,"imageIndex":7},{"imageOffset":35476,"symbol":"JLI_Launch","symbolLocation":5520,"imageIndex":7},{"imageOffset":15284,"symbol":"main","symbolLocation":404,"imageIndex":8},{"imageOffset":50524,"symbol":"apple_main","symbolLocation":88,"imageIndex":7},{"imageOffset":29292,"symbol":"_pthread_start","symbolLocation":148,"imageIndex":10},{"imageOffset":8332,"symbol":"thread_start","symbolLocation":8,"imageIndex":10}]},{"id":1766448,"frames":[{"imageOffset":21104,"symbol":"__psynch_cvwait","symbolLocation":8,"imageIndex":0},{"imageOffset":30780,"symbol":"_pthread_cond_wait","symbolLocation":1236,"imageIndex":10},{"imageOffset":8664360,"symbol":"os::PlatformMonitor::wait(long)","symbolLocation":152,"imageIndex":11},{"imageOffset":8423528,"symbol":"Monitor::wait(long long)","symbolLocation":128,"imageIndex":11},{"imageOffset":10056684,"symbol":"Threads::destroy_vm()","symbolLocation":100,"imageIndex":11},{"imageOffset":5626168,"symbol":"jni_DestroyJavaVM","symbolLocation":212,"imageIndex":11},{"imageOffset":39928,"symbol":"JavaMain","symbolLocation":3156,"imageIndex":7},{"imageOffset":48356,"symbol":"ThreadJavaMain","symbolLocation":12,"imageIndex":7},{"imageOffset":29292,"symbol":"_pthread_start","symbolLocation":148,"imageIndex":10},{"imageOffset":8332,"symbol":"thread_start","symbolLocation":8,"imageIndex":10}]},{"id":1766449,"name":"Java: GC Thread#0","frames":[{"imageOffset":6380,"symbol":"semaphore_wait_trap","symbolLocation":8,"imageIndex":0},{"imageOffset":9198916,"symbol":"OSXSemaphore::wait()","symbolLocation":24,"imageIndex":11},{"imageOffset":10707264,"symbol":"WorkerThread::run()","symbolLocation":84,"imageIndex":11},{"imageOffset":10027656,"symbol":"Thread::call_run()","symbolLocation":224,"imageIndex":11},{"imageOffset":8631860,"symbol":"thread_native_entry(Thread*)","symbolLocation":344,"imageIndex":11},{"imageOffset":29292,"symbol":"_pthread_start","symbolLocation":148,"imageIndex":10},{"imageOffset":8332,"symbol":"thread_start","symbolLocation":8,"imageIndex":10}]},{"id":1766450,"name":"Java: G1 Main Marker","frames":[{"imageOffset":21104,"symbol":"__psynch_cvwait","symbolLocation":8,"imageIndex":0},{"imageOffset":30780,"symbol":"_pthread_cond_wait","symbolLocation":1236,"imageIndex":10},{"imageOffset":8664360,"symbol":"os::PlatformMonitor::wait(long)","symbolLocation":152,"imageIndex":11},{"imageOffset":8423376,"symbol":"Monitor::wait_without_safepoint_check(long long)","symbolLocation":48,"imageIndex":11},{"imageOffset":4124296,"symbol":"G1ConcurrentMarkThread::run_service()","symbolLocation":156,"imageIndex":11},{"imageOffset":3102556,"symbol":"ConcurrentGCThread::run()","symbolLocation":36,"imageIndex":11},{"imageOffset":10027656,"symbol":"Thread::call_run()","symbolLocation":224,"imageIndex":11},{"imageOffset":8631860,"symbol":"thread_native_entry(Thread*)","symbolLocation":344,"imageIndex":11},{"imageOffset":29292,"symbol":"_pthread_start","symbolLocation":148,"imageIndex":10},{"imageOffset":8332,"symbol":"thread_start","symbolLocation":8,"imageIndex":10}]},{"id":1766451,"name":"Java: G1 Conc#0","frames":[{"imageOffset":6380,"symbol":"semaphore_wait_trap","symbolLocation":8,"imageIndex":0},{"imageOffset":9198916,"symbol":"OSXSemaphore::wait()","symbolLocation":24,"imageIndex":11},{"imageOffset":10707264,"symbol":"WorkerThread::run()","symbolLocation":84,"imageIndex":11},{"imageOffset":10027656,"symbol":"Thread::call_run()","symbolLocation":224,"imageIndex":11},{"imageOffset":8631860,"symbol":"thread_native_entry(Thread*)","symbolLocation":344,"imageIndex":11},{"imageOffset":29292,"symbol":"_pthread_start","symbolLocation":148,"imageIndex":10},{"imageOffset":8332,"symbol":"thread_start","symbolLocation":8,"imageIndex":10}]},{"id":1766452,"name":"Java: G1 Refine#0","frames":[{"imageOffset":6380,"symbol":"semaphore_wait_trap","symbolLocation":8,"imageIndex":0},{"imageOffset":9198916,"symbol":"OSXSemaphore::wait()","symbolLocation":24,"imageIndex":11},{"imageOffset":4135496,"symbol":"G1PrimaryConcurrentRefineThread::wait_for_completed_buffers()","symbolLocation":24,"imageIndex":11},{"imageOffset":4134324,"symbol":"G1ConcurrentRefineThread::run_service()","symbolLocation":64,"imageIndex":11},{"imageOffset":3102556,"symbol":"ConcurrentGCThread::run()","symbolLocation":36,"imageIndex":11},{"imageOffset":10027656,"symbol":"Thread::call_run()","symbolLocation":224,"imageIndex":11},{"imageOffset":8631860,"symbol":"thread_native_entry(Thread*)","symbolLocation":344,"imageIndex":11},{"imageOffset":29292,"symbol":"_pthread_start","symbolLocation":148,"imageIndex":10},{"imageOffset":8332,"symbol":"thread_start","symbolLocation":8,"imageIndex":10}]},{"id":1766453,"name":"Java: G1 Service","frames":[{"imageOffset":21104,"symbol":"__psynch_cvwait","symbolLocation":8,"imageIndex":0},{"imageOffset":30780,"symbol":"_pthread_cond_wait","symbolLocation":1236,"imageIndex":10},{"imageOffset":8664472,"symbol":"os::PlatformMonitor::wait(long)","symbolLocation":264,"imageIndex":11},{"imageOffset":8423376,"symbol":"Monitor::wait_without_safepoint_check(long long)","symbolLocation":48,"imageIndex":11},{"imageOffset":4436160,"symbol":"G1ServiceThread::wait_for_task()","symbolLocation":172,"imageIndex":11},{"imageOffset":4436672,"symbol":"G1ServiceThread::run_service()","symbolLocation":40,"imageIndex":11},{"imageOffset":3102556,"symbol":"ConcurrentGCThread::run()","symbolLocation":36,"imageIndex":11},{"imageOffset":10027656,"symbol":"Thread::call_run()","symbolLocation":224,"imageIndex":11},{"imageOffset":8631860,"symbol":"thread_native_entry(Thread*)","symbolLocation":344,"imageIndex":11},{"imageOffset":29292,"symbol":"_pthread_start","symbolLocation":148,"imageIndex":10},{"imageOffset":8332,"symbol":"thread_start","symbolLocation":8,"imageIndex":10}]},{"id":1766454,"name":"Java: VM Thread","frames":[{"imageOffset":21104,"symbol":"__psynch_cvwait","symbolLocation":8,"imageIndex":0},{"imageOffset":30780,"symbol":"_pthread_cond_wait","symbolLocation":1236,"imageIndex":10},{"imageOffset":8664472,"symbol":"os::PlatformMonitor::wait(long)","symbolLocation":264,"imageIndex":11},{"imageOffset":8423376,"symbol":"Monitor::wait_without_safepoint_check(long long)","symbolLocation":48,"imageIndex":11},{"imageOffset":10555508,"symbol":"VMThread::wait_for_operation()","symbolLocation":468,"imageIndex":11},{"imageOffset":10551572,"symbol":"VMThread::run()","symbolLocation":188,"imageIndex":11},{"imageOffset":10027656,"symbol":"Thread::call_run()","symbolLocation":224,"imageIndex":11},{"imageOffset":8631860,"symbol":"thread_native_entry(Thread*)","symbolLocation":344,"imageIndex":11},{"imageOffset":29292,"symbol":"_pthread_start","symbolLocation":148,"imageIndex":10},{"imageOffset":8332,"symbol":"thread_start","symbolLocation":8,"imageIndex":10}]},{"id":1766455,"frames":[{"imageOffset":8312,"symbol":"start_wqthread","symbolLocation":0,"imageIndex":10}]},{"id":1766456,"frames":[{"imageOffset":8312,"symbol":"start_wqthread","symbolLocation":0,"imageIndex":10}]},{"id":1766457,"name":"Java: Reference Handler","frames":[{"imageOffset":21104,"symbol":"__psynch_cvwait","symbolLocation":8,"imageIndex":0},{"imageOffset":30780,"symbol":"_pthread_cond_wait","symbolLocation":1236,"imageIndex":10},{"imageOffset":8664360,"symbol":"os::PlatformMonitor::wait(long)","symbolLocation":152,"imageIndex":11},{"imageOffset":8423528,"symbol":"Monitor::wait(long long)","symbolLocation":128,"imageIndex":11},{"imageOffset":5880848,"symbol":"JVM_WaitForReferencePendingList","symbolLocation":204,"imageIndex":11},{"imageOffset":5223254700,"imageIndex":12},{"imageOffset":5223241032,"imageIndex":12},{"imageOffset":5223241032,"imageIndex":12},{"imageOffset":5223170368,"imageIndex":12},{"imageOffset":5126904,"symbol":"JavaCalls::call_helper(JavaValue*, methodHandle const&, JavaCallArguments*, JavaThread*)","symbolLocation":908,"imageIndex":11},{"imageOffset":5122836,"symbol":"JavaCalls::call_virtual(JavaValue*, Klass*, Symbol*, Symbol*, JavaCallArguments*, JavaThread*)","symbolLocation":284,"imageIndex":11},{"imageOffset":5123040,"symbol":"JavaCalls::call_virtual(JavaValue*, Handle, Klass*, Symbol*, Symbol*, JavaThread*)","symbolLocation":100,"imageIndex":11},{"imageOffset":5871052,"symbol":"thread_entry(JavaThread*, JavaThread*)","symbolLocation":156,"imageIndex":11},{"imageOffset":10034484,"symbol":"JavaThread::thread_main_inner()","symbolLocation":272,"imageIndex":11},{"imageOffset":10027656,"symbol":"Thread::call_run()","symbolLocation":224,"imageIndex":11},{"imageOffset":8631860,"symbol":"thread_native_entry(Thread*)","symbolLocation":344,"imageIndex":11},{"imageOffset":29292,"symbol":"_pthread_start","symbolLocation":148,"imageIndex":10},{"imageOffset":8332,"symbol":"thread_start","symbolLocation":8,"imageIndex":10}]},{"id":1766458,"name":"Java: Finalizer","frames":[{"imageOffset":21104,"symbol":"__psynch_cvwait","symbolLocation":8,"imageIndex":0},{"imageOffset":30780,"symbol":"_pthread_cond_wait","symbolLocation":1236,"imageIndex":10},{"imageOffset":8660980,"symbol":"os::PlatformEvent::park()","symbolLocation":164,"imageIndex":11},{"imageOffset":8535896,"symbol":"ObjectMonitor::wait(long, bool, JavaThread*)","symbolLocation":1736,"imageIndex":11},{"imageOffset":9833084,"symbol":"ObjectSynchronizer::wait(Handle, long, JavaThread*)","symbolLocation":272,"imageIndex":11},{"imageOffset":5802020,"symbol":"JVM_MonitorWait","symbolLocation":368,"imageIndex":11},{"imageOffset":5223254700,"imageIndex":12},{"imageOffset":5223241032,"imageIndex":12},{"imageOffset":5223241032,"imageIndex":12},{"imageOffset":5223241032,"imageIndex":12},{"imageOffset":5223241032,"imageIndex":12},{"imageOffset":5223240448,"imageIndex":12},{"imageOffset":5223240448,"imageIndex":12},{"imageOffset":5223170368,"imageIndex":12},{"imageOffset":5126904,"symbol":"JavaCalls::call_helper(JavaValue*, methodHandle const&, JavaCallArguments*, JavaThread*)","symbolLocation":908,"imageIndex":11},{"imageOffset":5122836,"symbol":"JavaCalls::call_virtual(JavaValue*, Klass*, Symbol*, Symbol*, JavaCallArguments*, JavaThread*)","symbolLocation":284,"imageIndex":11},{"imageOffset":5123040,"symbol":"JavaCalls::call_virtual(JavaValue*, Handle, Klass*, Symbol*, Symbol*, JavaThread*)","symbolLocation":100,"imageIndex":11},{"imageOffset":5871052,"symbol":"thread_entry(JavaThread*, JavaThread*)","symbolLocation":156,"imageIndex":11},{"imageOffset":10034484,"symbol":"JavaThread::thread_main_inner()","symbolLocation":272,"imageIndex":11},{"imageOffset":10027656,"symbol":"Thread::call_run()","symbolLocation":224,"imageIndex":11},{"imageOffset":8631860,"symbol":"thread_native_entry(Thread*)","symbolLocation":344,"imageIndex":11},{"imageOffset":29292,"symbol":"_pthread_start","symbolLocation":148,"imageIndex":10},{"imageOffset":8332,"symbol":"thread_start","symbolLocation":8,"imageIndex":10}]},{"id":1766459,"name":"Java: Signal Dispatcher","frames":[{"imageOffset":6380,"symbol":"semaphore_wait_trap","symbolLocation":8,"imageIndex":0},{"imageOffset":9198916,"symbol":"OSXSemaphore::wait()","symbolLocation":24,"imageIndex":11},{"imageOffset":9283212,"symbol":"os::signal_wait()","symbolLocation":180,"imageIndex":11},{"imageOffset":8612568,"symbol":"signal_thread_entry(JavaThread*, JavaThread*)","symbolLocation":76,"imageIndex":11},{"imageOffset":10034484,"symbol":"JavaThread::thread_main_inner()","symbolLocation":272,"imageIndex":11},{"imageOffset":10027656,"symbol":"Thread::call_run()","symbolLocation":224,"imageIndex":11},{"imageOffset":8631860,"symbol":"thread_native_entry(Thread*)","symbolLocation":344,"imageIndex":11},{"imageOffset":29292,"symbol":"_pthread_start","symbolLocation":148,"imageIndex":10},{"imageOffset":8332,"symbol":"thread_start","symbolLocation":8,"imageIndex":10}]},{"id":1766460,"name":"Java: Service Thread","frames":[{"imageOffset":21104,"symbol":"__psynch_cvwait","symbolLocation":8,"imageIndex":0},{"imageOffset":30780,"symbol":"_pthread_cond_wait","symbolLocation":1236,"imageIndex":10},{"imageOffset":8664360,"symbol":"os::PlatformMonitor::wait(long)","symbolLocation":152,"imageIndex":11},{"imageOffset":8423376,"symbol":"Monitor::wait_without_safepoint_check(long long)","symbolLocation":48,"imageIndex":11},{"imageOffset":9201652,"symbol":"ServiceThread::service_thread_entry(JavaThread*, JavaThread*)","symbolLocation":484,"imageIndex":11},{"imageOffset":10034484,"symbol":"JavaThread::thread_main_inner()","symbolLocation":272,"imageIndex":11},{"imageOffset":10027656,"symbol":"Thread::call_run()","symbolLocation":224,"imageIndex":11},{"imageOffset":8631860,"symbol":"thread_native_entry(Thread*)","symbolLocation":344,"imageIndex":11},{"imageOffset":29292,"symbol":"_pthread_start","symbolLocation":148,"imageIndex":10},{"imageOffset":8332,"symbol":"thread_start","symbolLocation":8,"imageIndex":10}]},{"id":1766461,"name":"Java: Monitor Deflation Thread","frames":[{"imageOffset":21104,"symbol":"__psynch_cvwait","symbolLocation":8,"imageIndex":0},{"imageOffset":30780,"symbol":"_pthread_cond_wait","symbolLocation":1236,"imageIndex":10},{"imageOffset":8664472,"symbol":"os::PlatformMonitor::wait(long)","symbolLocation":264,"imageIndex":11},{"imageOffset":8423376,"symbol":"Monitor::wait_without_safepoint_check(long long)","symbolLocation":48,"imageIndex":11},{"imageOffset":8377668,"symbol":"MonitorDeflationThread::monitor_deflation_thread_entry(JavaThread*, JavaThread*)","symbolLocation":152,"imageIndex":11},{"imageOffset":10034484,"symbol":"JavaThread::thread_main_inner()","symbolLocation":272,"imageIndex":11},{"imageOffset":10027656,"symbol":"Thread::call_run()","symbolLocation":224,"imageIndex":11},{"imageOffset":8631860,"symbol":"thread_native_entry(Thread*)","symbolLocation":344,"imageIndex":11},{"imageOffset":29292,"symbol":"_pthread_start","symbolLocation":148,"imageIndex":10},{"imageOffset":8332,"symbol":"thread_start","symbolLocation":8,"imageIndex":10}]},{"id":1766462,"name":"Java: C2 CompilerThread0","frames":[{"imageOffset":8892696,"symbol":"PhaseChaitin::elide_copy(Node*, int, Block*, Node_List&, Node_List&, bool)","symbolLocation":1052,"imageIndex":11},{"imageOffset":8892176,"symbol":"PhaseChaitin::elide_copy(Node*, int, Block*, Node_List&, Node_List&, bool)","symbolLocation":532,"imageIndex":11},{"imageOffset":8898348,"symbol":"PhaseChaitin::post_allocate_copy_removal()","symbolLocation":2984,"imageIndex":11},{"imageOffset":2468828,"symbol":"PhaseChaitin::Register_Allocate()","symbolLocation":1976,"imageIndex":11},{"imageOffset":2956700,"symbol":"Compile::Code_Gen()","symbolLocation":760,"imageIndex":11},{"imageOffset":2947772,"symbol":"Compile::Compile(ciEnv*, ciMethod*, int, Options, DirectiveSet*)","symbolLocation":2944,"imageIndex":11},{"imageOffset":2341384,"symbol":"C2Compiler::compile_method(ciEnv*, ciMethod*, int, bool, DirectiveSet*)","symbolLocation":308,"imageIndex":11},{"imageOffset":3017028,"symbol":"CompileBroker::invoke_compiler_on_method(CompileTask*)","symbolLocation":1280,"imageIndex":11},{"imageOffset":3015596,"symbol":"CompileBroker::compiler_thread_loop()","symbolLocation":1112,"imageIndex":11},{"imageOffset":10034484,"symbol":"JavaThread::thread_main_inner()","symbolLocation":272,"imageIndex":11},{"imageOffset":10027656,"symbol":"Thread::call_run()","symbolLocation":224,"imageIndex":11},{"imageOffset":8631860,"symbol":"thread_native_entry(Thread*)","symbolLocation":344,"imageIndex":11},{"imageOffset":29292,"symbol":"_pthread_start","symbolLocation":148,"imageIndex":10},{"imageOffset":8332,"symbol":"thread_start","symbolLocation":8,"imageIndex":10}]},{"id":1766463,"name":"Java: C1 CompilerThread0","frames":[{"imageOffset":21104,"symbol":"__psynch_cvwait","symbolLocation":8,"imageIndex":0},{"imageOffset":30780,"symbol":"_pthread_cond_wait","symbolLocation":1236,"imageIndex":10},{"imageOffset":8664472,"symbol":"os::PlatformMonitor::wait(long)","symbolLocation":264,"imageIndex":11},{"imageOffset":8423528,"symbol":"Monitor::wait(long long)","symbolLocation":128,"imageIndex":11},{"imageOffset":3001860,"symbol":"CompileQueue::get(CompilerThread*)","symbolLocation":728,"imageIndex":11},{"imageOffset":3015144,"symbol":"CompileBroker::compiler_thread_loop()","symbolLocation":660,"imageIndex":11},{"imageOffset":10034484,"symbol":"JavaThread::thread_main_inner()","symbolLocation":272,"imageIndex":11},{"imageOffset":10027656,"symbol":"Thread::call_run()","symbolLocation":224,"imageIndex":11},{"imageOffset":8631860,"symbol":"thread_native_entry(Thread*)","symbolLocation":344,"imageIndex":11},{"imageOffset":29292,"symbol":"_pthread_start","symbolLocation":148,"imageIndex":10},{"imageOffset":8332,"symbol":"thread_start","symbolLocation":8,"imageIndex":10}]},{"id":1766464,"name":"Java: Sweeper thread","frames":[{"imageOffset":21104,"symbol":"__psynch_cvwait","symbolLocation":8,"imageIndex":0},{"imageOffset":30780,"symbol":"_pthread_cond_wait","symbolLocation":1236,"imageIndex":10},{"imageOffset":8664472,"symbol":"os::PlatformMonitor::wait(long)","symbolLocation":264,"imageIndex":11},{"imageOffset":8423376,"symbol":"Monitor::wait_without_safepoint_check(long long)","symbolLocation":48,"imageIndex":11},{"imageOffset":9797036,"symbol":"NMethodSweeper::sweeper_loop()","symbolLocation":148,"imageIndex":11},{"imageOffset":10034484,"symbol":"JavaThread::thread_main_inner()","symbolLocation":272,"imageIndex":11},{"imageOffset":10027656,"symbol":"Thread::call_run()","symbolLocation":224,"imageIndex":11},{"imageOffset":8631860,"symbol":"thread_native_entry(Thread*)","symbolLocation":344,"imageIndex":11},{"imageOffset":29292,"symbol":"_pthread_start","symbolLocation":148,"imageIndex":10},{"imageOffset":8332,"symbol":"thread_start","symbolLocation":8,"imageIndex":10}]},{"id":1766465,"name":"Java: Notification Thread","frames":[{"imageOffset":21104,"symbol":"__psynch_cvwait","symbolLocation":8,"imageIndex":0},{"imageOffset":30780,"symbol":"_pthread_cond_wait","symbolLocation":1236,"imageIndex":10},{"imageOffset":8664360,"symbol":"os::PlatformMonitor::wait(long)","symbolLocation":152,"imageIndex":11},{"imageOffset":8423376,"symbol":"Monitor::wait_without_safepoint_check(long long)","symbolLocation":48,"imageIndex":11},{"imageOffset":8506548,"symbol":"NotificationThread::notification_thread_entry(JavaThread*, JavaThread*)","symbolLocation":164,"imageIndex":11},{"imageOffset":10034484,"symbol":"JavaThread::thread_main_inner()","symbolLocation":272,"imageIndex":11},{"imageOffset":10027656,"symbol":"Thread::call_run()","symbolLocation":224,"imageIndex":11},{"imageOffset":8631860,"symbol":"thread_native_entry(Thread*)","symbolLocation":344,"imageIndex":11},{"imageOffset":29292,"symbol":"_pthread_start","symbolLocation":148,"imageIndex":10},{"imageOffset":8332,"symbol":"thread_start","symbolLocation":8,"imageIndex":10}]},{"id":1766466,"name":"Java: VM Periodic Task Thread","frames":[{"imageOffset":21104,"symbol":"__psynch_cvwait","symbolLocation":8,"imageIndex":0},{"imageOffset":30780,"symbol":"_pthread_cond_wait","symbolLocation":1236,"imageIndex":10},{"imageOffset":8664472,"symbol":"os::PlatformMonitor::wait(long)","symbolLocation":264,"imageIndex":11},{"imageOffset":8423376,"symbol":"Monitor::wait_without_safepoint_check(long long)","symbolLocation":48,"imageIndex":11},{"imageOffset":8504896,"symbol":"WatcherThread::sleep() const","symbolLocation":176,"imageIndex":11},{"imageOffset":8505068,"symbol":"WatcherThread::run()","symbolLocation":76,"imageIndex":11},{"imageOffset":10027656,"symbol":"Thread::call_run()","symbolLocation":224,"imageIndex":11},{"imageOffset":8631860,"symbol":"thread_native_entry(Thread*)","symbolLocation":344,"imageIndex":11},{"imageOffset":29292,"symbol":"_pthread_start","symbolLocation":148,"imageIndex":10},{"imageOffset":8332,"symbol":"thread_start","symbolLocation":8,"imageIndex":10}]},{"id":1766467,"name":"Java: Common-Cleaner","frames":[{"imageOffset":21104,"symbol":"__psynch_cvwait","symbolLocation":8,"imageIndex":0},{"imageOffset":30780,"symbol":"_pthread_cond_wait","symbolLocation":1236,"imageIndex":10},{"imageOffset":8662508,"symbol":"Parker::park(bool, long)","symbolLocation":484,"imageIndex":11},{"imageOffset":10251448,"symbol":"Unsafe_Park(JNIEnv_*, _jobject*, unsigned char, long)","symbolLocation":336,"imageIndex":11},{"imageOffset":5223254700,"imageIndex":12},{"imageOffset":5223241032,"imageIndex":12},{"imageOffset":5223241032,"imageIndex":12},{"imageOffset":5223242024,"imageIndex":12},{"imageOffset":5223241032,"imageIndex":12},{"imageOffset":5223240448,"imageIndex":12},{"imageOffset":5223240448,"imageIndex":12},{"imageOffset":5223242376,"imageIndex":12},{"imageOffset":5223241032,"imageIndex":12},{"imageOffset":5223170368,"imageIndex":12},{"imageOffset":5126904,"symbol":"JavaCalls::call_helper(JavaValue*, methodHandle const&, JavaCallArguments*, JavaThread*)","symbolLocation":908,"imageIndex":11},{"imageOffset":5122836,"symbol":"JavaCalls::call_virtual(JavaValue*, Klass*, Symbol*, Symbol*, JavaCallArguments*, JavaThread*)","symbolLocation":284,"imageIndex":11},{"imageOffset":5123040,"symbol":"JavaCalls::call_virtual(JavaValue*, Handle, Klass*, Symbol*, Symbol*, JavaThread*)","symbolLocation":100,"imageIndex":11},{"imageOffset":5871052,"symbol":"thread_entry(JavaThread*, JavaThread*)","symbolLocation":156,"imageIndex":11},{"imageOffset":10034484,"symbol":"JavaThread::thread_main_inner()","symbolLocation":272,"imageIndex":11},{"imageOffset":10027656,"symbol":"Thread::call_run()","symbolLocation":224,"imageIndex":11},{"imageOffset":8631860,"symbol":"thread_native_entry(Thread*)","symbolLocation":344,"imageIndex":11},{"imageOffset":29292,"symbol":"_pthread_start","symbolLocation":148,"imageIndex":10},{"imageOffset":8332,"symbol":"thread_start","symbolLocation":8,"imageIndex":10}]},{"id":1766482,"queue":"com.apple.root.utility-qos","frames":[{"imageOffset":110952,"symbol":"dyld4::PrebuiltLoader::contains(dyld4::RuntimeState&, void const*, void const**, unsigned long long*, unsigned char*) const","symbolLocation":116,"imageIndex":9},{"imageOffset":110884,"symbol":"dyld4::PrebuiltLoader::contains(dyld4::RuntimeState&, void const*, void const**, unsigned long long*, unsigned char*) const","symbolLocation":48,"imageIndex":9},{"imageOffset":153880,"symbol":"invocation function for block in dyld4::APIs::findImageContaining(void*)","symbolLocation":124,"imageIndex":9},{"imageOffset":43672,"symbol":"dyld4::RuntimeState::withLoadersReadLock(void () block_pointer)","symbolLocation":92,"imageIndex":9},{"imageOffset":149868,"symbol":"dyld4::APIs::findImageContaining(void*)","symbolLocation":132,"imageIndex":9},{"imageOffset":154460,"symbol":"dyld4::APIs::dlopen_from(char const*, int, void*)","symbolLocation":256,"imageIndex":9},{"imageOffset":60248,"imageIndex":13},{"imageOffset":16820,"symbol":"_dispatch_client_callout","symbolLocation":20,"imageIndex":14},{"imageOffset":23092,"symbol":"_dispatch_once_callout","symbolLocation":32,"imageIndex":14},{"imageOffset":60080,"imageIndex":13},{"imageOffset":59676,"imageIndex":13},{"imageOffset":9712,"symbol":"_dispatch_call_block_and_release","symbolLocation":32,"imageIndex":14},{"imageOffset":16820,"symbol":"_dispatch_client_callout","symbolLocation":20,"imageIndex":14},{"imageOffset":88580,"symbol":"_dispatch_root_queue_drain","symbolLocation":680,"imageIndex":14},{"imageOffset":90372,"symbol":"_dispatch_worker_thread2","symbolLocation":164,"imageIndex":14},{"imageOffset":13092,"symbol":"_pthread_wqthread","symbolLocation":228,"imageIndex":10},{"imageOffset":8320,"symbol":"start_wqthread","symbolLocation":8,"imageIndex":10}]},{"id":1766484,"name":"Java: AWT-Shutdown","frames":[{"imageOffset":21104,"symbol":"__psynch_cvwait","symbolLocation":8,"imageIndex":0},{"imageOffset":30780,"symbol":"_pthread_cond_wait","symbolLocation":1236,"imageIndex":10},{"imageOffset":8660980,"symbol":"os::PlatformEvent::park()","symbolLocation":164,"imageIndex":11},{"imageOffset":8535896,"symbol":"ObjectMonitor::wait(long, bool, JavaThread*)","symbolLocation":1736,"imageIndex":11},{"imageOffset":9833084,"symbol":"ObjectSynchronizer::wait(Handle, long, JavaThread*)","symbolLocation":272,"imageIndex":11},{"imageOffset":5802020,"symbol":"JVM_MonitorWait","symbolLocation":368,"imageIndex":11},{"imageOffset":5223254700,"imageIndex":12},{"imageOffset":5223241032,"imageIndex":12},{"imageOffset":5223241032,"imageIndex":12},{"imageOffset":5223241032,"imageIndex":12},{"imageOffset":5223242376,"imageIndex":12},{"imageOffset":5223170368,"imageIndex":12},{"imageOffset":5126904,"symbol":"JavaCalls::call_helper(JavaValue*, methodHandle const&, JavaCallArguments*, JavaThread*)","symbolLocation":908,"imageIndex":11},{"imageOffset":5122836,"symbol":"JavaCalls::call_virtual(JavaValue*, Klass*, Symbol*, Symbol*, JavaCallArguments*, JavaThread*)","symbolLocation":284,"imageIndex":11},{"imageOffset":5123040,"symbol":"JavaCalls::call_virtual(JavaValue*, Handle, Klass*, Symbol*, Symbol*, JavaThread*)","symbolLocation":100,"imageIndex":11},{"imageOffset":5871052,"symbol":"thread_entry(JavaThread*, JavaThread*)","symbolLocation":156,"imageIndex":11},{"imageOffset":10034484,"symbol":"JavaThread::thread_main_inner()","symbolLocation":272,"imageIndex":11},{"imageOffset":10027656,"symbol":"Thread::call_run()","symbolLocation":224,"imageIndex":11},{"imageOffset":8631860,"symbol":"thread_native_entry(Thread*)","symbolLocation":344,"imageIndex":11},{"imageOffset":29292,"symbol":"_pthread_start","symbolLocation":148,"imageIndex":10},{"imageOffset":8332,"symbol":"thread_start","symbolLocation":8,"imageIndex":10}]},{"id":1766493,"name":"com.apple.NSEventThread","frames":[{"imageOffset":6320,"symbol":"mach_msg_trap","symbolLocation":8,"imageIndex":0},{"imageOffset":7456,"symbol":"mach_msg","symbolLocation":76,"imageIndex":0},{"imageOffset":545472,"symbol":"__CFRunLoopServiceMachPort","symbolLocation":372,"imageIndex":1},{"imageOffset":538480,"symbol":"__CFRunLoopRun","symbolLocation":1180,"imageIndex":1},{"imageOffset":535348,"symbol":"CFRunLoopRunSpecific","symbolLocation":600,"imageIndex":1},{"imageOffset":1761536,"symbol":"_NSEventThread","symbolLocation":196,"imageIndex":3},{"imageOffset":29292,"symbol":"_pthread_start","symbolLocation":148,"imageIndex":10},{"imageOffset":8332,"symbol":"thread_start","symbolLocation":8,"imageIndex":10}]},{"id":1766497,"name":"Java: Java2D Queue Flusher","frames":[{"imageOffset":21104,"symbol":"__psynch_cvwait","symbolLocation":8,"imageIndex":0},{"imageOffset":30780,"symbol":"_pthread_cond_wait","symbolLocation":1236,"imageIndex":10},{"imageOffset":8661560,"symbol":"os::PlatformEvent::park(long)","symbolLocation":428,"imageIndex":11},{"imageOffset":8535424,"symbol":"ObjectMonitor::wait(long, bool, JavaThread*)","symbolLocation":1264,"imageIndex":11},{"imageOffset":9833084,"symbol":"ObjectSynchronizer::wait(Handle, long, JavaThread*)","symbolLocation":272,"imageIndex":11},{"imageOffset":5802020,"symbol":"JVM_MonitorWait","symbolLocation":368,"imageIndex":11},{"imageOffset":5223254700,"imageIndex":12},{"imageOffset":5223241032,"imageIndex":12},{"imageOffset":5223241032,"imageIndex":12},{"imageOffset":5223242376,"imageIndex":12},{"imageOffset":5223170368,"imageIndex":12},{"imageOffset":5126904,"symbol":"JavaCalls::call_helper(JavaValue*, methodHandle const&, JavaCallArguments*, JavaThread*)","symbolLocation":908,"imageIndex":11},{"imageOffset":5122836,"symbol":"JavaCalls::call_virtual(JavaValue*, Klass*, Symbol*, Symbol*, JavaCallArguments*, JavaThread*)","symbolLocation":284,"imageIndex":11},{"imageOffset":5123040,"symbol":"JavaCalls::call_virtual(JavaValue*, Handle, Klass*, Symbol*, Symbol*, JavaThread*)","symbolLocation":100,"imageIndex":11},{"imageOffset":5871052,"symbol":"thread_entry(JavaThread*, JavaThread*)","symbolLocation":156,"imageIndex":11},{"imageOffset":10034484,"symbol":"JavaThread::thread_main_inner()","symbolLocation":272,"imageIndex":11},{"imageOffset":10027656,"symbol":"Thread::call_run()","symbolLocation":224,"imageIndex":11},{"imageOffset":8631860,"symbol":"thread_native_entry(Thread*)","symbolLocation":344,"imageIndex":11},{"imageOffset":29292,"symbol":"_pthread_start","symbolLocation":148,"imageIndex":10},{"imageOffset":8332,"symbol":"thread_start","symbolLocation":8,"imageIndex":10}]},{"id":1766498,"name":"Java: Java2D Disposer","frames":[{"imageOffset":21104,"symbol":"__psynch_cvwait","symbolLocation":8,"imageIndex":0},{"imageOffset":30780,"symbol":"_pthread_cond_wait","symbolLocation":1236,"imageIndex":10},{"imageOffset":8662528,"symbol":"Parker::park(bool, long)","symbolLocation":504,"imageIndex":11},{"imageOffset":10251448,"symbol":"Unsafe_Park(JNIEnv_*, _jobject*, unsigned char, long)","symbolLocation":336,"imageIndex":11},{"imageOffset":5223254700,"imageIndex":12},{"imageOffset":5223241032,"imageIndex":12},{"imageOffset":5223241032,"imageIndex":12},{"imageOffset":5223242024,"imageIndex":12},{"imageOffset":5223241032,"imageIndex":12},{"imageOffset":5223241032,"imageIndex":12},{"imageOffset":5223242376,"imageIndex":12},{"imageOffset":5223241032,"imageIndex":12},{"imageOffset":5223240448,"imageIndex":12},{"imageOffset":5223240448,"imageIndex":12},{"imageOffset":5223242376,"imageIndex":12},{"imageOffset":5223170368,"imageIndex":12},{"imageOffset":5126904,"symbol":"JavaCalls::call_helper(JavaValue*, methodHandle const&, JavaCallArguments*, JavaThread*)","symbolLocation":908,"imageIndex":11},{"imageOffset":5122836,"symbol":"JavaCalls::call_virtual(JavaValue*, Klass*, Symbol*, Symbol*, JavaCallArguments*, JavaThread*)","symbolLocation":284,"imageIndex":11},{"imageOffset":5123040,"symbol":"JavaCalls::call_virtual(JavaValue*, Handle, Klass*, Symbol*, Symbol*, JavaThread*)","symbolLocation":100,"imageIndex":11},{"imageOffset":5871052,"symbol":"thread_entry(JavaThread*, JavaThread*)","symbolLocation":156,"imageIndex":11},{"imageOffset":10034484,"symbol":"JavaThread::thread_main_inner()","symbolLocation":272,"imageIndex":11},{"imageOffset":10027656,"symbol":"Thread::call_run()","symbolLocation":224,"imageIndex":11},{"imageOffset":8631860,"symbol":"thread_native_entry(Thread*)","symbolLocation":344,"imageIndex":11},{"imageOffset":29292,"symbol":"_pthread_start","symbolLocation":148,"imageIndex":10},{"imageOffset":8332,"symbol":"thread_start","symbolLocation":8,"imageIndex":10}]},{"id":1766500,"name":"Java: Active Reference Queue Daemon","frames":[{"imageOffset":21104,"symbol":"__psynch_cvwait","symbolLocation":8,"imageIndex":0},{"imageOffset":30780,"symbol":"_pthread_cond_wait","symbolLocation":1236,"imageIndex":10},{"imageOffset":8662528,"symbol":"Parker::park(bool, long)","symbolLocation":504,"imageIndex":11},{"imageOffset":10251448,"symbol":"Unsafe_Park(JNIEnv_*, _jobject*, unsigned char, long)","symbolLocation":336,"imageIndex":11},{"imageOffset":5223254700,"imageIndex":12},{"imageOffset":5223241032,"imageIndex":12},{"imageOffset":5223241032,"imageIndex":12},{"imageOffset":5223242024,"imageIndex":12},{"imageOffset":5223241032,"imageIndex":12},{"imageOffset":5223241032,"imageIndex":12},{"imageOffset":5223242376,"imageIndex":12},{"imageOffset":5223241032,"imageIndex":12},{"imageOffset":5223240448,"imageIndex":12},{"imageOffset":5223240448,"imageIndex":12},{"imageOffset":5223240448,"imageIndex":12},{"imageOffset":5223170368,"imageIndex":12},{"imageOffset":5126904,"symbol":"JavaCalls::call_helper(JavaValue*, methodHandle const&, JavaCallArguments*, JavaThread*)","symbolLocation":908,"imageIndex":11},{"imageOffset":5122836,"symbol":"JavaCalls::call_virtual(JavaValue*, Klass*, Symbol*, Symbol*, JavaCallArguments*, JavaThread*)","symbolLocation":284,"imageIndex":11},{"imageOffset":5123040,"symbol":"JavaCalls::call_virtual(JavaValue*, Handle, Klass*, Symbol*, Symbol*, JavaThread*)","symbolLocation":100,"imageIndex":11},{"imageOffset":5871052,"symbol":"thread_entry(JavaThread*, JavaThread*)","symbolLocation":156,"imageIndex":11},{"imageOffset":10034484,"symbol":"JavaThread::thread_main_inner()","symbolLocation":272,"imageIndex":11},{"imageOffset":10027656,"symbol":"Thread::call_run()","symbolLocation":224,"imageIndex":11},{"imageOffset":8631860,"symbol":"thread_native_entry(Thread*)","symbolLocation":344,"imageIndex":11},{"imageOffset":29292,"symbol":"_pthread_start","symbolLocation":148,"imageIndex":10},{"imageOffset":8332,"symbol":"thread_start","symbolLocation":8,"imageIndex":10}]},{"id":1766503,"name":"Java: RequestProcessor queue manager","frames":[{"imageOffset":21104,"symbol":"__psynch_cvwait","symbolLocation":8,"imageIndex":0},{"imageOffset":30780,"symbol":"_pthread_cond_wait","symbolLocation":1236,"imageIndex":10},{"imageOffset":8661560,"symbol":"os::PlatformEvent::park(long)","symbolLocation":428,"imageIndex":11},{"imageOffset":8535424,"symbol":"ObjectMonitor::wait(long, bool, JavaThread*)","symbolLocation":1264,"imageIndex":11},{"imageOffset":9833084,"symbol":"ObjectSynchronizer::wait(Handle, long, JavaThread*)","symbolLocation":272,"imageIndex":11},{"imageOffset":5802020,"symbol":"JVM_MonitorWait","symbolLocation":368,"imageIndex":11},{"imageOffset":5223254700,"imageIndex":12},{"imageOffset":5223241032,"imageIndex":12},{"imageOffset":5223241032,"imageIndex":12},{"imageOffset":5223240448,"imageIndex":12},{"imageOffset":5223170368,"imageIndex":12},{"imageOffset":5126904,"symbol":"JavaCalls::call_helper(JavaValue*, methodHandle const&, JavaCallArguments*, JavaThread*)","symbolLocation":908,"imageIndex":11},{"imageOffset":5122836,"symbol":"JavaCalls::call_virtual(JavaValue*, Klass*, Symbol*, Symbol*, JavaCallArguments*, JavaThread*)","symbolLocation":284,"imageIndex":11},{"imageOffset":5123040,"symbol":"JavaCalls::call_virtual(JavaValue*, Handle, Klass*, Symbol*, Symbol*, JavaThread*)","symbolLocation":100,"imageIndex":11},{"imageOffset":5871052,"symbol":"thread_entry(JavaThread*, JavaThread*)","symbolLocation":156,"imageIndex":11},{"imageOffset":10034484,"symbol":"JavaThread::thread_main_inner()","symbolLocation":272,"imageIndex":11},{"imageOffset":10027656,"symbol":"Thread::call_run()","symbolLocation":224,"imageIndex":11},{"imageOffset":8631860,"symbol":"thread_native_entry(Thread*)","symbolLocation":344,"imageIndex":11},{"imageOffset":29292,"symbol":"_pthread_start","symbolLocation":148,"imageIndex":10},{"imageOffset":8332,"symbol":"thread_start","symbolLocation":8,"imageIndex":10}]},{"id":1766512,"name":"Java: GC Thread#1","frames":[{"imageOffset":6380,"symbol":"semaphore_wait_trap","symbolLocation":8,"imageIndex":0},{"imageOffset":9198916,"symbol":"OSXSemaphore::wait()","symbolLocation":24,"imageIndex":11},{"imageOffset":10707264,"symbol":"WorkerThread::run()","symbolLocation":84,"imageIndex":11},{"imageOffset":10027656,"symbol":"Thread::call_run()","symbolLocation":224,"imageIndex":11},{"imageOffset":8631860,"symbol":"thread_native_entry(Thread*)","symbolLocation":344,"imageIndex":11},{"imageOffset":29292,"symbol":"_pthread_start","symbolLocation":148,"imageIndex":10},{"imageOffset":8332,"symbol":"thread_start","symbolLocation":8,"imageIndex":10}]},{"id":1766513,"name":"Java: GC Thread#2","frames":[{"imageOffset":6380,"symbol":"semaphore_wait_trap","symbolLocation":8,"imageIndex":0},{"imageOffset":9198916,"symbol":"OSXSemaphore::wait()","symbolLocation":24,"imageIndex":11},{"imageOffset":10707264,"symbol":"WorkerThread::run()","symbolLocation":84,"imageIndex":11},{"imageOffset":10027656,"symbol":"Thread::call_run()","symbolLocation":224,"imageIndex":11},{"imageOffset":8631860,"symbol":"thread_native_entry(Thread*)","symbolLocation":344,"imageIndex":11},{"imageOffset":29292,"symbol":"_pthread_start","symbolLocation":148,"imageIndex":10},{"imageOffset":8332,"symbol":"thread_start","symbolLocation":8,"imageIndex":10}]},{"id":1766514,"name":"Java: GC Thread#3","frames":[{"imageOffset":6380,"symbol":"semaphore_wait_trap","symbolLocation":8,"imageIndex":0},{"imageOffset":9198916,"symbol":"OSXSemaphore::wait()","symbolLocation":24,"imageIndex":11},{"imageOffset":10707264,"symbol":"WorkerThread::run()","symbolLocation":84,"imageIndex":11},{"imageOffset":10027656,"symbol":"Thread::call_run()","symbolLocation":224,"imageIndex":11},{"imageOffset":8631860,"symbol":"thread_native_entry(Thread*)","symbolLocation":344,"imageIndex":11},{"imageOffset":29292,"symbol":"_pthread_start","symbolLocation":148,"imageIndex":10},{"imageOffset":8332,"symbol":"thread_start","symbolLocation":8,"imageIndex":10}]},{"id":1766515,"name":"Java: GC Thread#4","frames":[{"imageOffset":6380,"symbol":"semaphore_wait_trap","symbolLocation":8,"imageIndex":0},{"imageOffset":9198916,"symbol":"OSXSemaphore::wait()","symbolLocation":24,"imageIndex":11},{"imageOffset":10707264,"symbol":"WorkerThread::run()","symbolLocation":84,"imageIndex":11},{"imageOffset":10027656,"symbol":"Thread::call_run()","symbolLocation":224,"imageIndex":11},{"imageOffset":8631860,"symbol":"thread_native_entry(Thread*)","symbolLocation":344,"imageIndex":11},{"imageOffset":29292,"symbol":"_pthread_start","symbolLocation":148,"imageIndex":10},{"imageOffset":8332,"symbol":"thread_start","symbolLocation":8,"imageIndex":10}]},{"id":1766516,"name":"Java: GC Thread#5","frames":[{"imageOffset":6380,"symbol":"semaphore_wait_trap","symbolLocation":8,"imageIndex":0},{"imageOffset":9198916,"symbol":"OSXSemaphore::wait()","symbolLocation":24,"imageIndex":11},{"imageOffset":10707264,"symbol":"WorkerThread::run()","symbolLocation":84,"imageIndex":11},{"imageOffset":10027656,"symbol":"Thread::call_run()","symbolLocation":224,"imageIndex":11},{"imageOffset":8631860,"symbol":"thread_native_entry(Thread*)","symbolLocation":344,"imageIndex":11},{"imageOffset":29292,"symbol":"_pthread_start","symbolLocation":148,"imageIndex":10},{"imageOffset":8332,"symbol":"thread_start","symbolLocation":8,"imageIndex":10}]},{"id":1766517,"name":"Java: GC Thread#6","frames":[{"imageOffset":6380,"symbol":"semaphore_wait_trap","symbolLocation":8,"imageIndex":0},{"imageOffset":9198916,"symbol":"OSXSemaphore::wait()","symbolLocation":24,"imageIndex":11},{"imageOffset":10707264,"symbol":"WorkerThread::run()","symbolLocation":84,"imageIndex":11},{"imageOffset":10027656,"symbol":"Thread::call_run()","symbolLocation":224,"imageIndex":11},{"imageOffset":8631860,"symbol":"thread_native_entry(Thread*)","symbolLocation":344,"imageIndex":11},{"imageOffset":29292,"symbol":"_pthread_start","symbolLocation":148,"imageIndex":10},{"imageOffset":8332,"symbol":"thread_start","symbolLocation":8,"imageIndex":10}]},{"id":1766518,"name":"Java: GC Thread#7","frames":[{"imageOffset":6380,"symbol":"semaphore_wait_trap","symbolLocation":8,"imageIndex":0},{"imageOffset":9198916,"symbol":"OSXSemaphore::wait()","symbolLocation":24,"imageIndex":11},{"imageOffset":10707264,"symbol":"WorkerThread::run()","symbolLocation":84,"imageIndex":11},{"imageOffset":10027656,"symbol":"Thread::call_run()","symbolLocation":224,"imageIndex":11},{"imageOffset":8631860,"symbol":"thread_native_entry(Thread*)","symbolLocation":344,"imageIndex":11},{"imageOffset":29292,"symbol":"_pthread_start","symbolLocation":148,"imageIndex":10},{"imageOffset":8332,"symbol":"thread_start","symbolLocation":8,"imageIndex":10}]},{"triggered":true,"id":1766531,"name":"Java: AWT-EventQueue-0","threadState":{"x":[{"value":0},{"value":0},{"value":0},{"value":0},{"value":0},{"value":0},{"value":0},{"value":0},{"value":3379110092826083510},{"value":3379110092826083510},{"value":503316480},{"value":1},{"value":10765747506},{"value":0},{"value":4363026721},{"value":8375620688,"symbolLocation":0,"symbol":"OBJC_CLASS_$___NSCFString"},{"value":18446744073709551604},{"value":8399503104},{"value":0},{"value":105553169676800},{"value":4379973952},{"value":8133940970,"objc-selector":"setCurrentContext:"},{"value":105553169676800},{"value":10762602464},{"value":8334217216},{"value":8334209024},{"value":10765748240},{"value":8133794523,"objc-selector":"display"},{"value":1}],"flavor":"ARM_THREAD_STATE64","lr":{"value":12123830941307854592},"cpsr":{"value":1610616832},"fp":{"value":10765748096},"sp":{"value":10765748016},"esr":{"value":4060086273,"description":"(Breakpoint) brk 1"},"pc":{"value":6938124032,"matchesCrashFrame":1},"far":{"value":10826940416}},"frames":[{"imageOffset":3690240,"symbol":"-[NSOpenGLContext setView:]","symbolLocation":228,"imageIndex":3},{"imageOffset":3690240,"symbol":"-[NSOpenGLContext setView:]","symbolLocation":228,"imageIndex":3},{"imageOffset":7386368,"symbol":"-[_NSOpenGLViewBackingLayer display]","symbolLocation":628,"imageIndex":3},{"imageOffset":134960,"symbol":"CA::Layer::display_if_needed(CA::Transaction*)","symbolLocation":780,"imageIndex":15},{"imageOffset":1496620,"symbol":"CA::Context::commit_transaction(CA::Transaction*, double, double*)","symbolLocation":464,"imageIndex":15},{"imageOffset":9420,"symbol":"CA::Transaction::commit()","symbolLocation":704,"imageIndex":15},{"imageOffset":5223254700,"imageIndex":12},{"imageOffset":5223240768,"imageIndex":12},{"imageOffset":5223241032,"imageIndex":12},{"imageOffset":5223242112,"imageIndex":12},{"imageOffset":5223241032,"imageIndex":12},{"imageOffset":5223241032,"imageIndex":12},{"imageOffset":5223242376,"imageIndex":12},{"imageOffset":5223242376,"imageIndex":12},{"imageOffset":5223241032,"imageIndex":12},{"imageOffset":5223240448,"imageIndex":12},{"imageOffset":5101947236,"imageIndex":12},{"imageOffset":5223240448,"imageIndex":12},{"imageOffset":5223241792,"imageIndex":12},{"imageOffset":5223241032,"imageIndex":12},{"imageOffset":5223241032,"imageIndex":12},{"imageOffset":5223241032,"imageIndex":12},{"imageOffset":5223241032,"imageIndex":12},{"imageOffset":5223241032,"imageIndex":12},{"imageOffset":5223241032,"imageIndex":12},{"imageOffset":5223170368,"imageIndex":12},{"imageOffset":5126904,"symbol":"JavaCalls::call_helper(JavaValue*, methodHandle const&, JavaCallArguments*, JavaThread*)","symbolLocation":908,"imageIndex":11},{"imageOffset":5122836,"symbol":"JavaCalls::call_virtual(JavaValue*, Klass*, Symbol*, Symbol*, JavaCallArguments*, JavaThread*)","symbolLocation":284,"imageIndex":11},{"imageOffset":5123040,"symbol":"JavaCalls::call_virtual(JavaValue*, Handle, Klass*, Symbol*, Symbol*, JavaThread*)","symbolLocation":100,"imageIndex":11},{"imageOffset":5871052,"symbol":"thread_entry(JavaThread*, JavaThread*)","symbolLocation":156,"imageIndex":11},{"imageOffset":10034484,"symbol":"JavaThread::thread_main_inner()","symbolLocation":272,"imageIndex":11},{"imageOffset":10027656,"symbol":"Thread::call_run()","symbolLocation":224,"imageIndex":11},{"imageOffset":8631860,"symbol":"thread_native_entry(Thread*)","symbolLocation":344,"imageIndex":11},{"imageOffset":29292,"symbol":"_pthread_start","symbolLocation":148,"imageIndex":10},{"imageOffset":8332,"symbol":"thread_start","symbolLocation":8,"imageIndex":10}]},{"id":1766534,"frames":[{"imageOffset":8312,"symbol":"start_wqthread","symbolLocation":0,"imageIndex":10}]},{"id":1766535,"frames":[{"imageOffset":8312,"symbol":"start_wqthread","symbolLocation":0,"imageIndex":10}]}],
  "usedImages" : [
  {
    "source" : "P",
    "arch" : "arm64e",
    "base" : 6888099840,
    "size" : 229376,
    "uuid" : "d5cdde7e-7037-3a70-9ce4-9e3847da6ce0",
    "path" : "\/usr\/lib\/system\/libsystem_kernel.dylib",
    "name" : "libsystem_kernel.dylib"
  },
  {
    "source" : "P",
    "arch" : "arm64e",
    "base" : 6888652800,
    "CFBundleShortVersionString" : "6.9",
    "CFBundleIdentifier" : "com.apple.CoreFoundation",
    "size" : 5533696,
    "uuid" : "1643f3d9-7049-37e4-b844-c0c0a42bde3b",
    "path" : "\/System\/Library\/Frameworks\/CoreFoundation.framework\/Versions\/A\/CoreFoundation",
    "name" : "CoreFoundation",
    "CFBundleVersion" : "1866"
  },
  {
    "source" : "P",
    "arch" : "arm64e",
    "base" : 7036043264,
    "CFBundleShortVersionString" : "2.1.1",
    "CFBundleIdentifier" : "com.apple.HIToolbox",
    "size" : 3358720,
    "uuid" : "7112f6f2-a77e-39db-83c8-f627b8a49df8",
    "path" : "\/System\/Library\/Frameworks\/Carbon.framework\/Versions\/A\/Frameworks\/HIToolbox.framework\/Versions\/A\/HIToolbox",
    "name" : "HIToolbox"
  },
  {
    "source" : "P",
    "arch" : "arm64e",
    "base" : 6934433792,
    "CFBundleShortVersionString" : "6.9",
    "CFBundleIdentifier" : "com.apple.AppKit",
    "size" : 15437824,
    "uuid" : "07bc3607-552a-3eb9-a941-4911c589dbc5",
    "path" : "\/System\/Library\/Frameworks\/AppKit.framework\/Versions\/C\/AppKit",
    "name" : "AppKit",
    "CFBundleVersion" : "2113.60.148"
  },
  {
    "source" : "P",
    "arch" : "arm64",
    "base" : 4382195712,
    "size" : 32768,
    "uuid" : "856e2938-924e-3a37-ba51-f4e36166890e",
    "path" : "\/Users\/USER\/Library\/Java\/JavaVirtualMachines\/openjdk-19\/Contents\/Home\/lib\/libosxapp.dylib",
    "name" : "libosxapp.dylib"
  },
  {
    "source" : "P",
    "arch" : "arm64",
    "base" : 4424990720,
    "size" : 770048,
    "uuid" : "ddd1182a-2838-3f1b-827b-0f71cd65f46b",
    "path" : "\/Users\/USER\/Library\/Java\/JavaVirtualMachines\/openjdk-19\/Contents\/Home\/lib\/libawt_lwawt.dylib",
    "name" : "libawt_lwawt.dylib"
  },
  {
    "source" : "P",
    "arch" : "arm64e",
    "base" : 6904397824,
    "CFBundleShortVersionString" : "6.9",
    "CFBundleIdentifier" : "com.apple.Foundation",
    "size" : 4136960,
    "uuid" : "59ff845a-05b2-3b64-ae06-6ef1d6aee83e",
    "path" : "\/System\/Library\/Frameworks\/Foundation.framework\/Versions\/C\/Foundation",
    "name" : "Foundation",
    "CFBundleVersion" : "1866"
  },
  {
    "source" : "P",
    "arch" : "arm64",
    "base" : 4363223040,
    "size" : 81920,
    "uuid" : "8f8cedd4-4d12-3aec-9e5e-8d26fd66b393",
    "path" : "\/Users\/USER\/Library\/Java\/JavaVirtualMachines\/openjdk-19\/Contents\/Home\/lib\/libjli.dylib",
    "name" : "libjli.dylib"
  },
  {
    "source" : "P",
    "arch" : "arm64",
    "base" : 4362977280,
    "size" : 16384,
    "uuid" : "46d333fe-7faa-3cc8-8390-eef50d4a85f3",
    "path" : "\/Users\/USER\/Library\/Java\/JavaVirtualMachines\/openjdk-19\/Contents\/Home\/bin\/java",
    "name" : "java"
  },
  {
    "source" : "P",
    "arch" : "arm64e",
    "base" : 4364025856,
    "size" : 393216,
    "uuid" : "75627683-a780-32ad-ae34-cf86dd23a26b",
    "path" : "\/usr\/lib\/dyld",
    "name" : "dyld"
  },
  {
    "source" : "P",
    "arch" : "arm64e",
    "base" : 6888329216,
    "size" : 53248,
    "uuid" : "f32ff902-ba43-30b0-ad43-a2a8a9ff69fe",
    "path" : "\/usr\/lib\/system\/libsystem_pthread.dylib",
    "name" : "libsystem_pthread.dylib"
  },
  {
    "source" : "P",
    "arch" : "arm64",
    "base" : 4382752768,
    "size" : 11747328,
    "uuid" : "02462c0d-ed7e-3b56-a84b-830fc2e1b189",
    "path" : "\/Users\/USER\/Library\/Java\/JavaVirtualMachines\/openjdk-19\/Contents\/Home\/lib\/server\/libjvm.dylib",
    "name" : "libjvm.dylib"
  },
  {
    "size" : 0,
    "source" : "A",
    "base" : 0,
    "uuid" : "00000000-0000-0000-0000-000000000000"
  },
  {
    "source" : "P",
    "arch" : "arm64e",
    "base" : 8523591680,
    "CFBundleShortVersionString" : "19.5.1",
    "CFBundleIdentifier" : "com.apple.opengl",
    "size" : 69632,
    "uuid" : "f538e7dd-90fa-3ea1-ae05-3db879ca7c6a",
    "path" : "\/System\/Library\/Frameworks\/OpenGL.framework\/Versions\/A\/OpenGL",
    "name" : "OpenGL",
    "CFBundleVersion" : "19.5.1"
  },
  {
    "source" : "P",
    "arch" : "arm64e",
    "base" : 6886490112,
    "size" : 290816,
    "uuid" : "035d23d9-4cb5-3759-9059-1f7878f89fee",
    "path" : "\/usr\/lib\/system\/libdispatch.dylib",
    "name" : "libdispatch.dylib"
  },
  {
    "source" : "P",
    "arch" : "arm64e",
    "base" : 7005032448,
    "CFBundleShortVersionString" : "1.11",
    "CFBundleIdentifier" : "com.apple.QuartzCore",
    "size" : 3325952,
    "uuid" : "05d8e758-d3ea-3942-bbe3-668a95692658",
    "path" : "\/System\/Library\/Frameworks\/QuartzCore.framework\/Versions\/A\/QuartzCore",
    "name" : "QuartzCore",
    "CFBundleVersion" : "964.35"
  }
],
  "sharedCache" : {
  "base" : 6885031936,
  "size" : 3144712192,
  "uuid" : "f8ab76fb-504c-32dd-a5db-15a1f2f028f8"
},
  "vmSummary" : "ReadOnly portion of Libraries: Total=816.8M resident=0K(0%) swapped_out_or_unallocated=816.8M(100%)\nWritable regions: Total=2.0G written=0K(0%) resident=0K(0%) swapped_out=0K(0%) unallocated=2.0G(100%)\n\n                                VIRTUAL   REGION \nREGION TYPE                        SIZE    COUNT (non-coalesced) \n===========                     =======  ======= \nAccelerate framework               512K        4 \nActivity Tracing                   256K        1 \nCG backing stores                 4800K        8 \nCG image                          8304K        4 \nColorSync                          608K       26 \nCoreAnimation                      256K        9 \nCoreGraphics                        48K        3 \nCoreUI image data                 1104K        7 \nFoundation                         112K        2 \nImage IO                          1024K        1 \nKernel Alloc Once                   32K        1 \nMALLOC                           322.6M       57 \nMALLOC guard page                  192K       10 \nMALLOC_MEDIUM (reserved)         944.0M        8         reserved VM address space (unallocated)\nMALLOC_NANO (reserved)           384.0M        1         reserved VM address space (unallocated)\nSTACK GUARD                       1200K       39 \nStack                             73.8M       39 \nStack Guard                       56.3M       17 \nVM_ALLOCATE                        3.4G      339 \nVM_ALLOCATE (reserved)            42.0M        9         reserved VM address space (unallocated)\n__AUTH                            1819K      160 \n__AUTH_CONST                      10.0M      312 \n__CTF                               756        1 \n__DATA                            10.3M      320 \n__DATA_CONST                      11.3M      332 \n__DATA_DIRTY                       604K      103 \n__FONT_DATA                          4K        1 \n__GLSLBUILTINS                    5176K        1 \n__LINKEDIT                       582.8M       23 \n__OBJC_CONST                      1290K      136 \n__OBJC_RO                         83.0M        1 \n__OBJC_RW                         3168K        1 \n__TEXT                           234.0M      348 \n__UNICODE                          592K        1 \ndyld private memory               1024K        1 \nmapped file                      298.0M       29 \nshared memory                      912K       16 \n===========                     =======  ======= \nTOTAL                              6.4G     2371 \nTOTAL, minus reserved VM space     5.0G     2371 \n",
  "legacyInfo" : {
  "threadTriggered" : {
    "name" : "Java: AWT-EventQueue-0"
  }
},
  "trialInfo" : {
  "rollouts" : [
    {
      "rolloutId" : "60356660bbe37970735c5624",
      "factorPackIds" : {

      },
      "deploymentId" : 240000027
    },
    {
      "rolloutId" : "60f8ddccefea4203d95cbeef",
      "factorPackIds" : {

      },
      "deploymentId" : 240000021
    }
  ],
  "experiments" : [

  ]
}
}

Model: MacBookPro18,1, BootROM 7459.141.1, proc 10:8:2 processors, 16 GB, SMC 
Graphics: Apple M1 Pro, Apple M1 Pro, Built-In
Display: Color LCD, 3456 x 2234 Retina, Main, MirrorOff, Online
Display: DELL S2721HS, 1920 x 1080 (1080p FHD - Full High Definition), MirrorOff, Online
Memory Module: LPDDR5
AirPort: Wi-Fi, wl0: Apr  6 2022 05:55:54 version 20.90.45.0.8.7.118 FWID 01-e7138ff2
Bluetooth: Version (null), 0 services, 0 devices, 0 incoming serial ports
Network Service: Wi-Fi, AirPort, en0
USB Device: USB31Bus
USB Device: USB31Bus
USB Device: USB3.1 Hub
USB Device: USB 10/100/1000 LAN
USB Device: USB2.0 HUB
USB Device: USB C Video Adaptor
USB Device: USB2.1 Hub
USB Device: USB Optical Mouse
USB Device: EPOMAKER TH96 ISO Pro
USB Device: USB31Bus
Thunderbolt Bus: MacBook Pro, Apple Inc.
Thunderbolt Bus: MacBook Pro, Apple Inc.
Thunderbolt Bus: MacBook Pro, Apple Inc.

```

Reply:
Interesting, does it happen with Java 18 or less?
In my case first thread works for glfw but not for awt, and glfw does not work without first thread.

By the way, just updated the master branch with scaling fixes for GLFW, please pull the changes too

Reply:
I confirm AWT breaks with java 17 or more. It works at least with java 14 or less.
This happens regardless of using first thread or not.
This is probably caused by a compatibility problem with https://github.com/LWJGLX/lwjgl3-awt

Reply:
That is an interesting feedback ! How does it break ? JVM crash? Java exception ?

Reply:
... ouch. No idea to help you for now.
I can only say that JOGL 2.4-rc4 supports Apple M1.

Reply:
Just to clarify, it has nothing to do in this case with Apple M1, it breaks with AWT on Java 17 or later both in x86_64 and aarch64.
Will check if it happens in Windows and linux too.
Fortunately we still ship Gephi with Java 11 embedded

Reply:
CC @mbastian

Reply:
Hum note, that `-J-XstartOnFirstThread` on current Gephi doesn't work (JRE11). Netbeans Platform doesn't even start...

Reply:
Yeah that's no problem as it's only necessary for glfw but we dont use it (still viz engine supports it for potential embedded use).